# SOME DESCRIPTIVE TITLE.
# Copyright (C) YEAR THE PACKAGE'S COPYRIGHT HOLDER
# This file is distributed under the same license as the PACKAGE package.
# FIRST AUTHOR <EMAIL@ADDRESS>, YEAR.
#
#, fuzzy
msgid ""
msgstr ""
"Project-Id-Version: PACKAGE VERSION\n"
"Report-Msgid-Bugs-To: \n"
"POT-Creation-Date: 2022-04-15 00:46+0800\n"
"PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\n"
"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
"Language-Team: LANGUAGE <LL@li.org>\n"
"Language: \n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Plural-Forms: nplurals=2; plural=(n != 1);\n"

#: apps/backend/agent/manager.py:88
msgid "卸载Agent"
msgstr "Uninstall Agent"

#: apps/backend/agent/manager.py:97
msgid "卸载Proxy"
msgstr "Uninstall Proxy"

#: apps/backend/agent/manager.py:167
msgid "启动 NGINX 服务"
msgstr "Start NGINX service"

#: apps/backend/agent/manager.py:190
#, python-brace-format
msgid "托管 {plugin_name} 插件进程"
msgstr "Supervised {plugin_name} plugin process"

#: apps/backend/api/constants.py:117 apps/node_man/constants.py:377
#: apps/node_man/constants.py:688 apps/node_man/constants.py:724
#: apps/node_man/tools/job.py:53
msgid "执行成功"
msgstr "Succeed"

#: apps/backend/api/constants.py:118 apps/node_man/constants.py:376
#: apps/node_man/constants.py:687 apps/node_man/constants.py:723
msgid "正在执行"
msgstr "Running"

#: apps/backend/api/constants.py:119
msgid "Agent 状态异常"
msgstr "Agent state abnormal"

#: apps/backend/api/constants.py:120
msgid "进程正在运行中，无需启动"
msgstr "Process is running, no need to start"

#: apps/backend/api/constants.py:121
msgid "进程当前未运行，无需停止"
msgstr "The process is not currently running and does not need to be stopped"

#: apps/backend/api/constants.py:122
msgid "操作类型非法"
msgstr "Illegal operation type"

#: apps/backend/api/constants.py:123
msgid "进程启动失败（后置检查状态为未运行）"
msgstr "Process start failed (post-check status is not running)"

#: apps/backend/api/constants.py:124 apps/node_man/constants.py:378
#: apps/node_man/constants.py:685 apps/node_man/constants.py:721
#: apps/node_man/constants.py:725
msgid "执行失败"
msgstr "Failed"

#: apps/backend/api/errors.py:22
msgid "无响应"
msgstr "Not responding"

#: apps/backend/api/errors.py:23
#, python-brace-format
msgid "API({api_name})没有返回响应"
msgstr "API ({api_name}) returned no response"

#: apps/backend/api/errors.py:30
msgid "接口调用错误"
msgstr "API call error"

#: apps/backend/api/errors.py:31
#, python-brace-format
msgid "API({api_name})调用出错：{error_message}"
msgstr "Error calling API ({api_name}): {error_message}"

#: apps/backend/api/errors.py:38
msgid "Job轮询超时"
msgstr "Job poll timeout"

#: apps/backend/api/errors.py:39
#, python-brace-format
msgid "Job任务({job_instance_id})轮询超时"
msgstr "Job task ({job_instance_id}) polling timeout"

#: apps/backend/api/errors.py:46
msgid "GSE轮询超时"
msgstr "GSE poll timeout"

#: apps/backend/api/errors.py:47
#, python-brace-format
msgid "GSE任务({task_id})轮询超时"
msgstr "GSE task ({task_id}) poll timed out"

#: apps/backend/components/collections/agent_new/base.py:161
#, python-brace-format
msgid "所选安装通道「{name} 没有可用跳板机"
msgstr "The selected installation channel '{name} has no jumper available"

#: apps/backend/components/collections/agent_new/base.py:172
#: apps/backend/components/collections/agent_new/choose_access_point.py:317
#, python-brace-format
msgid "云区域 -> {bk_cloud_id} 下无存活的 Proxy"
msgstr "cloud area -> no live proxy under {bk_cloud_id}"

#: apps/backend/components/collections/agent_new/check_agent_status.py:42
msgid "Agent 状态【正常】"
msgstr "Agent status [normal]"

#: apps/backend/components/collections/agent_new/check_agent_status.py:44
msgid "Agent 状态【异常】"
msgstr "Agent status [abnormal]"

#: apps/backend/components/collections/agent_new/check_policy_gse_to_proxy.py:65
#: apps/backend/tests/components/collections/agent_new/test_check_policy_gse_to_proxy.py:59
msgid "请检查出口IP是否正确或策略是否开通"
msgstr "Please check whether the egress IP is correct or the policy is enabled"

#: apps/backend/components/collections/agent_new/check_policy_gse_to_proxy.py:78
#, python-brace-format
msgid ""
"请确保 GSE BT File Server 公网IP（{bt_server_ips}）到 Proxy 出口"
"IP（{proxy_outer_ip}）的 {btsvr_thrift_port}(tcp), {bt_port}-{tracker_port}"
"(tcp/udp) 访问策略正常。\n"
"另外请注意：\n"
"1. 可执行 curl ipinfo.io 确认「出口IP」\n"
"2. 不支持多台 Proxy 使用同一个「出口IP」\n"
"3. 请保证「出口IP」固定不变"
msgstr ""
"Please make sure the GSE BT File Server public IP ({bt_server_ips}) to the "
"Proxy exitIP({proxy_outer_ip}) {btsvr_thrift_port}(tcp), {bt_port}-"
"{tracker_port}(tcp/udp) access policy OK.\n"
"Also note:\n"
"1. Execute curl ipinfo.io to confirm the 'export IP'\n"
"2. Multiple proxies using the same 'exit IP' are not supported\n"
"3. Please ensure that the 'export IP' is fixed"

#: apps/backend/components/collections/agent_new/choose_access_point.py:68
msgid "选择接入点失败"
msgstr "Failed to select access point"

#: apps/backend/components/collections/agent_new/choose_access_point.py:70
#, python-brace-format
msgid "已选择 [{ap_name}] 作为本次安装的接入点"
msgstr ""
"[{ap_name}] has been selected as the access point for this installation"

#: apps/backend/components/collections/agent_new/choose_access_point.py:135
#, python-brace-format
msgid "连接至接入点 [{ap_name}] 的平均延迟为 {ping_time}ms"
msgstr ""
"The average latency to connect to access point [{ap_name}] is {ping_time}ms"

#: apps/backend/components/collections/agent_new/choose_access_point.py:142
#, python-brace-format
msgid ""
"主机 <--> 接入点网络情况检测结果：\n"
"{detect_logs_str}"
msgstr ""
"Host <--> Access point network condition test result:\n"
"{detect_logs_str}"

#: apps/backend/components/collections/agent_new/choose_access_point.py:150
msgid "自动选择接入点失败，接入点均 ping 不可达"
msgstr ""
"Failed to automatically select access points, all access points are ping "
"unreachable"

#: apps/backend/components/collections/agent_new/choose_access_point.py:350
#, python-brace-format
msgid ""
"已选择 Proxy 所在云区域「{bk_cloud_name}」[{bk_cloud_id}] 指定的接入点 "
"[{ap_name}]"
msgstr ""
"The access point specified by the cloud area {bk_cloud_name} [{bk_cloud_id}] "
"where the Proxy is located [{ap_name}]"

#: apps/backend/components/collections/agent_new/choose_access_point.py:425
msgid "自动选择接入点失败，请到全局配置新建接入点"
msgstr ""
"Failed to automatically select access point, please go to global "
"configuration to create a new access point"

#: apps/backend/components/collections/agent_new/choose_access_point.py:463
#, python-brace-format
msgid "当前主机已分配接入点 [{ap_name}]"
msgstr "The current host has assigned access point [{ap_name}]"

#: apps/backend/components/collections/agent_new/components.py:39
msgid "查询主机密码"
msgstr "Query host password"

#: apps/backend/components/collections/agent_new/components.py:45
msgid "选择接入点"
msgstr "Select access point"

#: apps/backend/components/collections/agent_new/components.py:51
msgid "注册主机到配置平台"
msgstr "register host to configuration platform"

#: apps/backend/components/collections/agent_new/components.py:57
#: apps/node_man/constants.py:282
msgid "安装"
msgstr "Install"

#: apps/backend/components/collections/agent_new/components.py:63
msgid "安装预设插件"
msgstr "Install preset plugin"

#: apps/backend/components/collections/agent_new/components.py:69
msgid "配置策略"
msgstr "Configure policy"

#: apps/backend/components/collections/agent_new/components.py:75
msgid "托管插件进程"
msgstr "Supervised plugin process"

#: apps/backend/components/collections/agent_new/components.py:81
#: apps/backend/plugin/manager.py:62
msgid "查询Agent状态"
msgstr "Query Agent Status"

#: apps/backend/components/collections/agent_new/components.py:87
msgid "重载Agent配置"
msgstr "Reload Agent Configuration"

#: apps/backend/components/collections/agent_new/components.py:93
#: apps/node_man/constants.py:283 apps/node_man/constants.py:461
msgid "重启"
msgstr "Restart"

#: apps/backend/components/collections/agent_new/components.py:99
msgid "更新主机进程状态"
msgstr "update host process status"

#: apps/backend/components/collections/agent_new/components.py:105
msgid "检查Agent状态"
msgstr "Check Agent Status"

#: apps/backend/components/collections/agent_new/components.py:111
msgid "下发升级包"
msgstr "Transfer upgrade package"

#: apps/backend/components/collections/agent_new/components.py:117
msgid "检测 GSE Server 到 Proxy 策略"
msgstr "Detecting GSE Server to Proxy Policy"

#: apps/backend/components/collections/agent_new/components.py:123
msgid "下发升级脚本命令"
msgstr "Send upgrade script command"

#: apps/backend/components/collections/agent_new/components.py:129
msgid "渲染并下发Agent配置"
msgstr "Render and deliver Agent configuration"

#: apps/backend/components/collections/agent_new/components.py:135
#: apps/backend/components/collections/agent_new/wait.py:22
msgid "等待"
msgstr "Wait"

#: apps/backend/components/collections/agent_new/components.py:141
msgid "执行脚本"
msgstr "Execute script"

#: apps/backend/components/collections/agent_new/components.py:147
msgid "分发文件"
msgstr "Distribution file"

#: apps/backend/components/collections/agent_new/components.py:153
msgid "分发配置"
msgstr "Distribution configuration"

#: apps/backend/components/collections/agent_new/components.py:159
msgid "下发文件到Proxy"
msgstr "Transfer file to Proxy"

#: apps/backend/components/collections/agent_new/components.py:165
msgid "更新安装信息"
msgstr "Update installation info"

#: apps/backend/components/collections/agent_new/configure_policy.py:54
msgid "到Gse和Nginx的策略配置成功"
msgstr "Policies to Gse and Nginx are configured successfully"

#: apps/backend/components/collections/agent_new/configure_policy.py:59
msgid "配置到Gse和Nginx的策略失败请联系节点管理维护人员"
msgstr ""
"Failed to configure policies to Gse and Nginx, please contact node "
"management and maintainers"

#: apps/backend/components/collections/agent_new/delegate_plugin_proc.py:98
#, python-brace-format
msgid ""
"更新 {proc_name} 进程信息：call GSE api -> update_proc_info with params: \n"
" {query_params})"
msgstr ""
"Update {proc_name} process info: call GSE api -> update_proc_info with "
"params: \"n {query_params})"

#: apps/backend/components/collections/agent_new/delegate_plugin_proc.py:127
#, python-brace-format
msgid "未能查询到进程更新结果，gse_proc_key -> {gse_proc_key}"
msgstr ""
"Failed to query the process update result, gse_proc_key -> {gse_proc_key}"

#: apps/backend/components/collections/agent_new/delegate_plugin_proc.py:136
#, python-brace-format
msgid ""
"GSE 更新 {proc_name} 进程信息失败，gse_proc_key -> {gse_proc_key}, "
"error_code -> {error_code}, error_msg -> {error_msg}"
msgstr ""
"GSE failed to update {proc_name} process information, gse_proc_key -> "
"{gse_proc_key},  error_code -> {error_code}, error_msg -> {error_msg}"

#: apps/backend/components/collections/agent_new/delegate_plugin_proc.py:151
#, python-brace-format
msgid "已成功更新 {proc_name} 进程信息"
msgstr "Successfully updated {proc_name} process information"

#: apps/backend/components/collections/agent_new/delegate_plugin_proc.py:180
#, python-brace-format
msgid "「{plugin_name}」不支持操作系统 -> {os_type}，跳过该步骤"
msgstr "{plugin_name} does not support OS -> {os_type}, skip this step"

#: apps/backend/components/collections/agent_new/delegate_plugin_proc.py:227
#, python-brace-format
msgid ""
"托管 {proc_name} 进程：call GSE api -> operate_proc_v2 with params: \n"
" {query_params}"
msgstr ""
"supervised {proc_name} process: call GSE api -> operate_proc_v2 with params: "
"\"n {query_params}"

#: apps/backend/components/collections/agent_new/delegate_plugin_proc.py:268
#, python-brace-format
msgid "未能查询到进程操作结果, task_id -> {task_id}, gse_proc_key -> {key}"
msgstr ""
"Failed to query process operation result, task_id -> {task_id}, gse_proc_key "
"->; {key}"

#: apps/backend/components/collections/agent_new/delegate_plugin_proc.py:280
#: apps/backend/components/collections/plugin.py:1197
msgid "GSE任务轮询超时"
msgstr "GSE task polling timeout"

#: apps/backend/components/collections/agent_new/delegate_plugin_proc.py:285
#, python-brace-format
msgid ""
"调用 GSE 接口异常：错误码 -> {error_code}「{error_code_alias}」, 错误信息 -> "
"{error_msg}"
msgstr ""
"Exception calling GSE interface: error code-> {error_code}"
"{error_code_alias}, error message->  {error_msg}"

#: apps/backend/components/collections/agent_new/get_agent_status.py:44
#, python-brace-format
msgid "期望的 Agent 状态为 {expect_status_alias}「{expect_status}」"
msgstr "The expected agent status is {expect_status_alias}[{expect_status}]"

#: apps/backend/components/collections/agent_new/get_agent_status.py:107
#, python-brace-format
msgid ""
"查询 GSE 得到主机 [{host_key}] Agent 状态 -> {status_alias}「{status}」, 版"
"本 -> {version}"
msgstr ""
"Query GSE to get host [{host_key}] Agent status -> {status_alias}[{status}], "
"version -> {version}"

#: apps/backend/components/collections/agent_new/get_agent_status.py:111
msgid "无"
msgstr "none"

#: apps/backend/components/collections/agent_new/get_agent_status.py:147
msgid "查询 GSE 超时"
msgstr "Query GSE timed out"

#: apps/backend/components/collections/agent_new/install.py:199
msgid "等待手动执行安装命令"
msgstr "waiting for manual install command"

#: apps/backend/components/collections/agent_new/install.py:299
#: apps/backend/utils/ssh.py:416 apps/exceptions.py:115
msgid "认证信息已过期, 请重装并填入认证信息"
msgstr ""
"The authentication information has expired, please reload and fill in the "
"authentication information"

#: apps/backend/components/collections/agent_new/install.py:321
msgid ""
"正在安装 Windows AGENT, 命令执行失败，请确认: \n"
"1. 检查文件共享相关服务，确认以下服务均已开启\n"
"    - Function Discovery Resource Publication\n"
"    - SSDP Discovery \n"
"    - UPnP Device Host\n"
"    - Server\n"
"    - NetLogon // 如果没有加入域，可以不启动这个\n"
"    - TCP/IP NetBIOS Helper\n"
"2. 开启网卡 Net BOIS \n"
"3. 开启文件共享 Net share \n"
"4. 检查防火墙是否有放开 139/135/445 端口 \n"
msgstr ""
"Installing Windows AGENT, command failed, please confirm:\n"
"1. Check file sharing related services and make sure the following services "
"are enabled\n"
" - Function Discovery Resource Publication\n"
" - SSDP Discovery \n"
" - UPnP Device Host\n"
"-Server\n"
" - NetLogon // If not joined to a domain, this can be disabled\n"
" - TCP/IP NetBIOS Helper\n"
"2. Enable the network card Net BOIS \n"
"3. Enable file sharing Net share \n"
"4. Check if the firewall has opened ports 139/135/445\n"

#: apps/backend/components/collections/agent_new/install.py:336
#: apps/backend/components/collections/agent_new/install.py:362
msgid "远程登录失败，等待自动重试，重试失败将打印异常信息。"
msgstr ""
"Remote login failed, waiting for automatic retry, if retry fails, an "
"exception message will be printed."

#: apps/backend/components/collections/agent_new/install.py:339
#: apps/backend/components/collections/agent_new/install.py:365
msgid "远程登录失败"
msgstr "remote login failed"

#: apps/backend/components/collections/agent_new/install.py:377
#, python-brace-format
msgid "主机的上游节点为: {upstream_nodes}"
msgstr "The upstream nodes of the host are: {upstream_nodes}"

#: apps/backend/components/collections/agent_new/install.py:381
#, python-brace-format
msgid "已选择 {inner_ip} 作为本次安装的跳板机"
msgstr "{inner_ip} has been selected as the springboard for this installation"

#: apps/backend/components/collections/agent_new/install.py:404
#, python-brace-format
msgid ""
"作业任务ID为[{job_instance_id}]，点击跳转到<a href=\"{link}\" target=\"_blank"
"\">[作业平台]</a>"
msgstr ""
"The JOB task ID is [{job_instance_id}], click to jump to <a href=\"{link}\" "
"target=\"_blank\">[Blueking JOB]</a>"

#: apps/backend/components/collections/agent_new/install.py:418
#, python-brace-format
msgid ""
"作业执行中，如果卡在这里较长时间，请检查：\n"
"1. P-Agent({host_inner_ip}) 到 Proxy({jump_server_ip}) 的 {download_port}、"
"{proxy_pass_port} 是否可连通。 \n"
"2. Proxy是否已正确完成所有安装步骤且状态正常。 \n"
"3. 点击上面链接跳转到作业平台查看任务执行情况。\n"
msgstr ""
"Job execution, if stuck here for a long time, please check:\n"
"1. P-Agent({host_inner_ip}) to {download_port} of Proxy({jump_server_ip}),"
"{proxy_pass_port} is reachable.\n"
"2. Whether the Proxy has completed all installation steps correctly and is "
"in a normal state.\n"
"3. Click the link above to jump to the job platform to view the task "
"execution status.\n"

#: apps/backend/components/collections/agent_new/install.py:484
#, python-brace-format
msgid "检测到目标机器已安装 curl -> {curl_path}，将通过 curl 下载依赖文件"
msgstr ""
"Detected that the target machine has curl -> {curl_path} installed, will "
"download the dependencies through curl"

#: apps/backend/components/collections/agent_new/install.py:500
#, python-brace-format
msgid ""
"推送下列文件到「{dest_dir}」\n"
"{filenames_str}"
msgstr "Push the following files to {dest_dir}\"n {filenames_str}"

#: apps/backend/components/collections/agent_new/install.py:583
msgid "安装超时"
msgstr "install timed out"

#: apps/backend/components/collections/agent_new/query_password.py:57
msgid ""
"若 OA TICKET 过期，请重新登录 OA 后再重试您的操作。请注意不要直接使用此任务中"
"的重试功能~"
msgstr ""
"If OA TICKET expires, please log in to OA again and try your operation "
"again. Please be careful not to use the retry function in this task directly "
"~"

#: apps/backend/components/collections/agent_new/query_password.py:82
msgid "当前主机验证类型无需查询密码"
msgstr "Current host verify without having to type a password query"

#: apps/backend/components/collections/agent_new/register_host.py:164
#, python-brace-format
msgid ""
"注册主机参数为:\n"
" {params}"
msgstr ""
"The registered host parameter is:\n"
"{params}"

#: apps/backend/components/collections/agent_new/register_host.py:173
#, python-brace-format
msgid ""
"注册主机失败：\n"
"{error_msgs}"
msgstr ""
"Failed to register host:\n"
"{error_msgs}"

#: apps/backend/components/collections/agent_new/register_host.py:233
msgid "查询CMDB主机失败，未在CMDB中查询到主机信息"
msgstr "Failed to query CMDB host, no host information was queried in CMDB"

#: apps/backend/components/collections/agent_new/register_host.py:241
#, python-brace-format
msgid ""
"主机期望注册到「{bk_biz_id}」[{bk_biz_name}]，但实际存在于业务「ID: "
"{actual_biz_id}」，请前往该业务进行安装"
msgstr ""
"The host expects to register with {bk_biz_id} [{bk_biz_name}], but actually "
"exists in the business ID:  {actual_biz_id}, please go to the business to "
"install"

#: apps/backend/components/collections/agent_new/register_host.py:284
msgid "该主机的登录认证信息已被清空，无法重试，请重新发起安装任务"
msgstr ""
"The login authentication information of this host has been cleared and "
"cannot be retried. Please re-initiate the installation task"

#: apps/backend/components/collections/agent_new/register_host.py:448
msgid "注册CMDB完成"
msgstr "Registration to CMDB completed"

#: apps/backend/components/collections/agent_new/update_process_status.py:49
#, python-brace-format
msgid "更新主机状态为{status}"
msgstr "Update host status to {status}"

#: apps/backend/components/collections/base.py:85
#, python-brace-format
msgid ""
"{act_name} 失败: {exc}，请先尝试查看错误日志进行处理，若无法解决，请联系管理"
"员处理"
msgstr ""
"{act_name} failed: {exc}, please try to check the error log to deal with it, "
"if it can't be solved, please contact the management manager to deal with"

#: apps/backend/components/collections/base.py:365
#, python-brace-format
msgid ""
"{act_name} 已终止，可整体重试/重试。（details: {revoke_sub_inst_id_set}）"
msgstr ""
"{act_name} terminated, retry/retry as a whole. (details: "
"{revoke_sub_inst_id_set})"

#: apps/backend/components/collections/base.py:388
#, python-brace-format
msgid "{act_name} 失败，请先尝试查看日志并处理，若无法解决，请联系管理员处理。"
msgstr ""
"{act_name} failed, please try to view the log and deal with it. If it cannot "
"be solved, please contact the administrator to deal with it."

#: apps/backend/components/collections/base.py:399
#, python-brace-format
msgid "{act_name} 成功"
msgstr "{act_name} succeeded"

#: apps/backend/components/collections/base.py:421
#, python-brace-format
msgid "开始 {act_name}."
msgstr "Start {act_name}."

#: apps/backend/components/collections/common/remote.py:37
msgid "SSH通道可用"
msgstr "SSH tunnel available"

#: apps/backend/components/collections/common/remote.py:38
msgid "SSH通道可用但登录信息有误"
msgstr "SSH channel available but wrong login information"

#: apps/backend/components/collections/common/remote.py:39
msgid "SSH通道不可用"
msgstr "SSH channel unavailable"

#: apps/backend/components/collections/common/remote.py:137
#, python-brace-format
msgid "当前登录用户为「{account}」，请确保该用户具有 sudo 权限"
msgstr ""
"The current logged-in user is {account}, please make sure this user has sudo "
"privileges"

#: apps/backend/components/collections/core.py:35
msgid "使用 ssh 会话执行命令"
msgstr "Execute command using ssh session"

#: apps/backend/components/collections/core.py:36
msgid "使用 wmiexe 会话执行命令"
msgstr "Execute command using wmiexe session"

#: apps/backend/components/collections/core.py:37
msgid "使用 job 执行命令"
msgstr "Execute command using job"

#: apps/backend/components/collections/core.py:38
msgid "查询密码"
msgstr "query password"

#: apps/backend/components/collections/job.py:148
#, python-brace-format
msgid ""
"下发配置文件 [{file_names}] 到目标机器路径 [{file_target_path}]，若下发失败，"
"请检查作业平台所部署的机器是否已安装AGENT"
msgstr ""
"Push config file [{file_names}] to the target machine path "
"[{file_target_path}], if the delivery fails,Please check whether the machine "
"deployed by the job platform has AGENT installed"

#: apps/backend/components/collections/job.py:152
#, python-brace-format
msgid "快速执行脚本 {script_name}"
msgstr "Fast execute scripts {script_name}"

#: apps/backend/components/collections/job.py:154
#, fuzzy
#| msgid "作业平台"
msgid "调用作业平台"
msgstr "call Blueking Job"

#: apps/backend/components/collections/job.py:165
#, python-brace-format
msgid ""
"{log}\n"
"作业任务ID为 [{job_instance_id}]，点击跳转到 <a href=\"{link}\" target="
"\"_blank\">[作业平台]</a>"
msgstr ""
"{log}\n"
"The JOB task ID is [{job_instance_id}], click to jump to <a href=\"{link}\" "
"target=\"_blank\">[Blueking JOB]</a>"

#: apps/backend/components/collections/job.py:236
#, python-brace-format
msgid ""
"作业执行状态 -> {status}「{ip_status_msg}」, 主机任务状态码 -> {error_code}"
"「{err_msg}」"
msgstr ""
"Job execution status -> {status} '{ip_status_msg}', host task status code -> {error_code} "
"'{err_msg}'"

#: apps/backend/components/collections/job.py:246
#, python-brace-format
msgid "作业平台执行失败: {err_msg}"
msgstr "Job task failed: {err_msg}"

#: apps/backend/components/collections/job.py:298
msgid "该步骤无需等待作业平台执行结果"
msgstr ""
"This step does not need to wait for the execution result of the job platform"

#: apps/backend/components/collections/job.py:355
msgid "作业平台执行任务超时"
msgstr "Job task execution task timed out"

#: apps/backend/components/collections/plugin.py:173
msgid "插件部署失败，重试次数 +1"
msgstr "Plugin deployment failed, retries +1"

#: apps/backend/components/collections/plugin.py:365
msgid "主机不存在或未同步"
msgstr "host does not exist or is not in sync"

#: apps/backend/components/collections/plugin.py:499
#, python-brace-format
msgid "{bk_host_id}:AGENT异常[{status}]，请尝试恢复AGENT状态或重装AGENT。"
msgstr ""
"{bk_host_id}: AGENT is abnormal [{status}], please try to restore AGENT "
"status or reinstall AGENT."

#: apps/backend/components/collections/plugin.py:842
msgid "主机[{}]在ip->[{}]上无可用端口"
msgstr "Host [{}] has no available port on ip->[{}]"

#: apps/backend/components/collections/plugin.py:980
#, python-brace-format
msgid ""
"无需渲染配置，直接进入下一步骤，subscription_instance_ids -> "
"{subscription_instance_ids}"
msgstr ""
"No rendering configuration, go directly to the next step, "
"subscription_instance_ids -> {subscription_instance_ids}"

#: apps/backend/components/collections/plugin.py:1235
#, python-brace-format
msgid "GSE任务查无结果, {task_id}, {key}"
msgstr "GSE task search no result, {task_id}, {key}"

#: apps/backend/components/collections/plugin.py:1376
#: apps/node_man/models.py:1788
msgid "策略"
msgstr "policy"

#: apps/backend/components/collections/plugin.py:1376
#: apps/backend/components/collections/plugin.py:1425
#: apps/node_man/models.py:501
msgid "订阅"
msgstr "subscribe"

#: apps/backend/components/collections/plugin.py:1381
#, python-brace-format
msgid "{sub_alias} -> {id}「{name}」 已启用，删除失败"
msgstr "{sub_alias} -> {id} {name} enabled, delete failed"

#: apps/backend/components/collections/plugin.py:1391
#, python-brace-format
msgid "{sub_alias} -> {id}「{name}」，已部署节点数 -> {proc_count}"
msgstr "{sub_alias} -> {id}{name}, number of deployed nodes -> {proc_count}"

#: apps/backend/components/collections/plugin.py:1400
#, python-brace-format
msgid "{sub_alias} -> {id}「{name}」 已部署节点数不为 0，跳过"
msgstr "{sub_alias} -> {id}{name} The number of deployed nodes is not 0, skip"

#: apps/backend/components/collections/plugin.py:1409
#, python-brace-format
msgid "{sub_alias} -> {id}「{name}」删除成功"
msgstr "{sub_alias} -> {id} {name} deleted successfully"

#: apps/backend/components/collections/plugin.py:1429
#, python-brace-format
msgid "{sub_alias} -> {id} 仅提供单次执行能力，保持{enable_alias}状态"
msgstr ""
"{sub_alias} -> {id} only provides single execution capability, keep "
"{enable_alias} state"

#: apps/backend/components/collections/plugin.py:1432
#: apps/backend/components/collections/plugin.py:1447
msgid "启用"
msgstr "Enable"

#: apps/backend/components/collections/plugin.py:1432
#: apps/backend/components/collections/plugin.py:1447
msgid "停用"
msgstr "Deactivate"

#: apps/backend/components/collections/plugin.py:1443
#, python-brace-format
msgid "{sub_alias} -> {id}「{name}」切换为：{enable_alias}"
msgstr "{sub_alias} -> {id} {name} switch to: {enable_alias}"

#: apps/backend/components/collections/subsubscription.py:97
#, python-brace-format
msgid "{node_name} 执行成功"
msgstr "{node_name} executed successfully"

#: apps/backend/constants.py:91
msgid "新安装"
msgstr "New install"

#: apps/backend/constants.py:92
msgid "版本变更"
msgstr "Version change"

#: apps/backend/constants.py:93
msgid "配置变更"
msgstr "Config change"

#: apps/backend/constants.py:94
msgid "进程数量不匹配"
msgstr "Number of processes does not match"

#: apps/backend/constants.py:95
msgid "进程状态异常"
msgstr "Process state abnormal"

#: apps/backend/constants.py:96
msgid "无需变更"
msgstr "No changes required"

#: apps/backend/constants.py:97
msgid "从范围中移除"
msgstr "Remove from scope"

#: apps/backend/constants.py:98
msgid "未同步的主机"
msgstr "Unsynced host"

#: apps/backend/constants.py:99
msgid "手动操作豁免"
msgstr "Manual Action Exemption"

#: apps/backend/exceptions.py:21
msgid "文件包不存在"
msgstr "Package does not exist"

#: apps/backend/exceptions.py:26 apps/node_man/exceptions.py:72
msgid "任务不存在"
msgstr "Task does not exist"

#: apps/backend/exceptions.py:31
msgid "停止调试失败"
msgstr "Failed to stop debugging"

#: apps/backend/exceptions.py:36 apps/backend/subscription/errors.py:120
msgid "插件包不存在"
msgstr "Plugin package does not exist"

#: apps/backend/exceptions.py:37
#, python-brace-format
msgid "插件包[{plugin_name}-{os_type}-{cpu_arch}]不存在"
msgstr "Plugin package [{plugin_name}-{os_type}-{cpu_arch}] does not exist"

#: apps/backend/exceptions.py:42
msgid "插件包状态变更错误"
msgstr "Plugin package state change error"

#: apps/backend/exceptions.py:47
msgid "插件包版本校验错误"
msgstr "Plugin package version check error"

#: apps/backend/exceptions.py:52
msgid "安装命令生成失败"
msgstr "Installation command generation failed"

#: apps/backend/exceptions.py:57
msgid "GSE敏感信息加密失败"
msgstr "GSE sensitive information encryption failed"

#: apps/backend/exceptions.py:62
msgid "插件解析错误"
msgstr "Plugin parsing error"

#: apps/backend/exceptions.py:67
msgid "归档插件包信息错误"
msgstr "Incorrect archive plugin package info"

#: apps/backend/healthz/checker/celery_checker.py:114
#: apps/backend/healthz/checker/supervisor_checker.py:86
msgid ", 进程启动时间小于10秒，请关注"
msgstr ""
", The process startup time is less than 10 seconds, please pay attention"

#: apps/backend/healthz/define.py:26
msgid "Redis 状态"
msgstr "Redis Status"

#: apps/backend/healthz/define.py:36
msgid "Redis 可读状态"
msgstr "Redis readable state"

#: apps/backend/healthz/define.py:46
msgid "Redis 可写状态"
msgstr "Redis writable state"

#: apps/backend/healthz/define.py:59
msgid "数据库状态"
msgstr "Database Status"

#: apps/backend/healthz/define.py:72
#, python-format
msgid "rabbitmq 队列长度, 单个队列阀值为%s条"
msgstr "Rabbitmq queue length, the threshold for a single queue is %s"

#: apps/backend/healthz/define.py:86
msgid "celery beat 状态"
msgstr "celery beat status"

#: apps/backend/healthz/define.py:96
msgid "celery worker 任务执行测试"
msgstr "celery worker task execution test"

#: apps/backend/healthz/define.py:106
msgid "celery worker 进程状态"
msgstr "celery worker process status"

#: apps/backend/healthz/define.py:122
msgid "Supervisor 自身状态"
msgstr "Supervisor own state"

#: apps/backend/healthz/define.py:132
msgid "Supervisor 进程状态"
msgstr "Supervisor process status"

#: apps/backend/healthz/define.py:142
msgid "Supervisor 逃逸进程"
msgstr "Supervisor escape process"

#: apps/backend/healthz/define.py:165
#, python-format
msgid "进程：%s未启动或连接不上"
msgstr "Process: %s is not started or cannot be connected"

#: apps/backend/healthz/define.py:166
#, python-format
msgid "确保进程：%s状态正常"
msgstr "Ensure process: %s is healthy"

#: apps/backend/periodic_tasks/check_zombie_sub_inst_record.py:59
#, python-brace-format
msgid ""
"\n"
"[{time_str} ERROR] 任务长时间处在执行状态，已强制失败"
msgstr ""
"\n"
"[{time_str} ERROR] The task has been in the execution state for a long time "
"and has been forced to fail"

#: apps/backend/plugin/handler.py:86
#, python-brace-format
msgid "文件不存在：file_path -> {file_path}"
msgstr "File does not exist: file_path -> {file_path}"

#: apps/backend/plugin/handler.py:88 apps/backend/plugin/handler.py:101
#: apps/backend/plugin/views.py:1230
msgid "上传文件MD5校验失败，请确认重试"
msgstr "Upload file MD5 verification failed, please confirm and try again"

#: apps/backend/plugin/manager.py:54
msgid "初始化进程状态"
msgstr "Initializing process state"

#: apps/backend/plugin/manager.py:71
#, python-brace-format
msgid "更新插件部署状态为{status}"
msgstr "Update plugin deployment status is {status}"

#: apps/backend/plugin/manager.py:77 apps/node_man/constants.py:829
msgid "下发安装包"
msgstr "Transfer installation package"

#: apps/backend/plugin/manager.py:81 apps/backend/plugin/manager.py:174
msgid "安装插件包"
msgstr "Install plugin package"

#: apps/backend/plugin/manager.py:85
msgid "卸载插件包"
msgstr "Uninstall plugin package"

#: apps/backend/plugin/manager.py:92
msgid "重置重试次数"
msgstr "Reset retries"

#: apps/backend/plugin/manager.py:99
msgid "分配可用端口"
msgstr "Allocate available ports"

#: apps/backend/plugin/manager.py:103
msgid "删除策略"
msgstr "Delete policy"

#: apps/backend/plugin/manager.py:107
msgid "切换订阅启用状态"
msgstr "Toggle subscription enabled state"

#: apps/backend/plugin/manager.py:113
msgid "渲染下发配置"
msgstr "Rendering config"

#: apps/backend/plugin/manager.py:124
msgid "调试插件"
msgstr "Debug plugin"

#: apps/backend/plugin/manager.py:133
msgid "停止调试插件"
msgstr "Stop debugging plugin"

#: apps/backend/plugin/manager.py:142
msgid "操作进程"
msgstr "Operation process"

#: apps/backend/plugin/manager.py:148 apps/node_man/constants.py:302
msgid "移除配置"
msgstr "Remove configuration"

#: apps/backend/plugin/manager.py:173
msgid "下发插件包"
msgstr "Transfer plugin package"

#: apps/backend/plugin/manager.py:175
msgid "下发配置文件"
msgstr "Push config file"

#: apps/backend/plugin/manager.py:176
msgid "执行插件调试进程"
msgstr "Execute plugin debugging process"

#: apps/backend/plugin/manager.py:177
msgid "结束调试并回收资源"
msgstr "End debugging and reclaim resources"

#: apps/backend/plugin/serializers.py:209
msgid "上传端计算的文件md5"
msgstr "File md5 calculated by uploader"

#: apps/backend/plugin/serializers.py:210
msgid "上传端提供的文件名"
msgstr "Filename provided by uploader"

#: apps/backend/plugin/serializers.py:217
msgid "本地文件路径"
msgstr "Local file path"

#: apps/backend/plugin/serializers.py:218
msgid "Nginx所计算的文件md5"
msgstr "File md5 calculated by Nginx"

#: apps/backend/plugin/serializers.py:224
msgid "文件下载url"
msgstr "File download url"

#: apps/backend/plugin/serializers.py:225
msgid "文件保存路径"
msgstr "File save path"

#: apps/backend/plugin/serializers.py:260
msgid "文件名称"
msgstr "Filename"

#: apps/backend/plugin/serializers.py:261
msgid "是否立即发布该插件"
msgstr "Do you want to publish this plugin now"

#: apps/backend/plugin/serializers.py:264
msgid "是否需要读取配置文件"
msgstr "Do you need to read configuration files"

#: apps/backend/plugin/serializers.py:265
msgid "是否可以覆盖已经存在的配置文件"
msgstr "Is it possible to overwrite an existing configuration file"

#: apps/backend/plugin/serializers.py:271
msgid "选择注册的插件包相对路径，缺省默认全选"
msgstr ""
"Select the relative path of the registered plugin package, the default "
"default is to select all"

#: apps/backend/plugin/tasks.py:68
msgid "插件包目录操作无权限"
msgstr "Plugin package directory operation without permission"

#: apps/backend/plugin/tasks.py:74
msgid "插件包上传记录不存在"
msgstr "Plugin package upload record does not exist"

#: apps/backend/plugin/tasks.py:80
msgid "插件包导入失败"
msgstr "Failed to import plugin package"

#: apps/backend/plugin/tools.py:58
#, python-brace-format
msgid "插件不存在: file_path -> {file_path}"
msgstr "Plugin does not exist: file_path -> {file_path}"

#: apps/backend/plugin/tools.py:72
#, python-brace-format
msgid "文件包含非法路径成员 -> {name}，请检查"
msgstr "File contains illegal path member -> {name}, please check"

#: apps/backend/plugin/tools.py:157
msgid "缺少project.yaml文件"
msgstr "Missing project.yaml file"

#: apps/backend/plugin/tools.py:173
msgid "project.yaml文件解析读取失败"
msgstr "Failed to parse and read the project.yaml file"

#: apps/backend/plugin/tools.py:203
msgid "project.yaml 文件信息缺失"
msgstr "project.yaml file information is missing"

#: apps/backend/plugin/tools.py:222
msgid "project.yaml 中 category 配置异常，请确认后重试"
msgstr ""
"The category configuration in project.yaml is abnormal, please confirm and "
"try again"

#: apps/backend/plugin/tools.py:235
msgid "新增插件"
msgstr "Add plugin"

#: apps/backend/plugin/tools.py:245
msgid "已有版本插件更新"
msgstr "Existing version plugin update"

#: apps/backend/plugin/tools.py:255
msgid "低版本插件仅支持导入"
msgstr "Lower version plugins only support import"

#: apps/backend/plugin/tools.py:258
msgid "更新插件版本"
msgstr "Update plugin version"

#: apps/backend/plugin/tools.py:269
#, python-brace-format
msgid ""
"文件路径 -> {pkg_absolute_path} 所在包解析版本为 -> {version}, 最新版本 -> "
"{release_version}, 更新校验失败"
msgstr ""
"File path -> {pkg_absolute_path} where the package resolution version -> "
"{version}, the latest version -> {release_version}, update verification "
"failed"

#: apps/backend/plugin/tools.py:290
#, python-brace-format
msgid "找不到需要导入的配置模板文件 -> {source_path}"
msgstr "Could not find config template file to import -> {source_path}"

#: apps/backend/plugin/tools.py:658
#, python-brace-format
msgid ""
"插件包保存错误，期望保存到 -> {package_target_path}, 实际保存到 -> "
"{storage_path}"
msgstr ""
"Error saving plugin package, expect to save to -> {package_target_path}, "
"actually save to -> {storage_path}"

#: apps/backend/plugin/views.py:104 apps/backend/plugin/views.py:813
msgid "找不到请求发布的文件，请确认后重试"
msgstr "The requested file cannot be found, please confirm and try again"

#: apps/backend/plugin/views.py:154
msgid "找不到请求的任务，请确认后重试"
msgstr "The requested task could not be found, please confirm and try again"

#: apps/backend/plugin/views.py:225
#, python-brace-format
msgid "ID{ids}的插件包未启用，无法执行更改状态操作"
msgstr ""
"Plugin package for ID {ids} is not enabled to perform change state operation"

#: apps/backend/plugin/views.py:638
msgid "调试任务准备中"
msgstr "Debugging task in preparation"

#: apps/backend/plugin/views.py:649
#, python-brace-format
msgid " 开始{name} "
msgstr "start {name}"

#: apps/backend/plugin/views.py:737 apps/backend/plugin/views.py:1298
msgid "请求任务不存在，请确认后重试"
msgstr "The requested task does not exist, please confirm and try again"

#: apps/backend/plugin/views.py:989 apps/backend/plugin/views.py:1114
#: apps/node_man/views/plugin_v2.py:157
#, python-brace-format
msgid "不存在ID为: {id} 的插件"
msgstr "Plugin with id: {id} does not exist"

#: apps/backend/plugin/views.py:1221 apps/backend/plugin/views.py:1265
#, python-brace-format
msgid "请求参数异常 [{err}]，请确认后重试"
msgstr ""
"The request parameter is abnormal [{err}], please confirm and try again"

#: apps/backend/plugin/views.py:1305
msgid "下载安全校验失败"
msgstr "Download security check failed"

#: apps/backend/subscription/errors.py:26
msgid "订阅不存在"
msgstr "Subscription does not exist"

#: apps/backend/subscription/errors.py:27
#, python-brace-format
msgid "订阅({subscription_id})不存在"
msgstr "Subscription ({subscription_id}) does not exist"

#: apps/backend/subscription/errors.py:34
msgid "Actions不能为None"
msgstr "Actions cannot be None"

#: apps/backend/subscription/errors.py:35
msgid "当scope存在时，actions不能空"
msgstr "actions cannot be empty when scope exists"

#: apps/backend/subscription/errors.py:42
msgid "订阅任务不存在"
msgstr "Subscription task does not exist"

#: apps/backend/subscription/errors.py:43
#, python-brace-format
msgid "订阅任务({task_id})不存在"
msgstr "Subscription task ({task_id}) does not exist"

#: apps/backend/subscription/errors.py:50
msgid "实例存在运行中任务"
msgstr "Instance has running tasks"

#: apps/backend/subscription/errors.py:51
msgid "实例存在运行中任务，避免重复下发"
msgstr "There are running tasks in the instance, avoid repeated delivery"

#: apps/backend/subscription/errors.py:58
msgid "配置文件渲染失败"
msgstr "Config template file rendering failed"

#: apps/backend/subscription/errors.py:59
#, python-brace-format
msgid "配置文件[{name}]渲染失败，原因：{msg}"
msgstr "Failed to render configuration file [{name}], reason: {msg}"

#: apps/backend/subscription/errors.py:66
msgid "Pipeline任务执行失败"
msgstr "Pipeline task execution failed"

#: apps/backend/subscription/errors.py:67
#, python-brace-format
msgid "Pipeline任务执行失败，原因：{msg}"
msgstr "Pipeline task execution failed, reason: {msg}"

#: apps/backend/subscription/errors.py:74
#: apps/backend/subscription/errors.py:75
msgid "Pipeline任务树解析失败"
msgstr "Pipeline task tree parsing failed"

#: apps/backend/subscription/errors.py:82
#: apps/backend/subscription/errors.py:83
msgid "订阅任务实例不存在"
msgstr "Subscription task instance does not exist"

#: apps/backend/subscription/errors.py:90
msgid "Pipeline任务重复执行"
msgstr "Pipeline task repeated"

#: apps/backend/subscription/errors.py:91
msgid "Pipeline任务已经开始执行，不能重复执行"
msgstr "Pipeline task has already started and cannot be repeated"

#: apps/backend/subscription/errors.py:98
msgid "订阅任务实例为空"
msgstr "Subscribe task instance is empty"

#: apps/backend/subscription/errors.py:99
msgid "订阅任务实例为空，不再创建订阅任务"
msgstr ""
"Subscription task instance is empty, no more subscription tasks will be "
"created"

#: apps/backend/subscription/errors.py:106
msgid "插件校验错误"
msgstr "Plugin validation error"

#: apps/backend/subscription/errors.py:107
#: apps/backend/subscription/errors.py:115
#: apps/backend/subscription/errors.py:121
#, python-brace-format
msgid "{msg}"
msgstr "{msg}"

#: apps/backend/subscription/errors.py:114
msgid "不允许同一个请求传不同目标"
msgstr "Do not allow the same request to pass different targets"

#: apps/backend/subscription/errors.py:128
msgid "订阅步骤不存在"
msgstr "Subscription step does not exist"

#: apps/backend/subscription/errors.py:129
#, python-brace-format
msgid "订阅步骤({step_id})不存在"
msgstr "Subscription step ({step_id}) does not exist"

#: apps/backend/subscription/errors.py:134
msgid "创建任务失败"
msgstr "Failed to create task"

#: apps/backend/subscription/errors.py:135
#, python-brace-format
msgid "创建任务失败: {err_msg}"
msgstr "Failed to create task: {err_msg}"

#: apps/backend/subscription/errors.py:140
msgid "订阅任务未准备就绪，请稍后再尝试查询"
msgstr "Subscription task not ready, please try query later"

#: apps/backend/subscription/errors.py:141
#, python-brace-format
msgid "订阅任务未准备就绪，请稍后再尝试查询。task_id_list: {task_id_list}"
msgstr ""
"Subscription task is not ready, please try querying later. task_id_list: "
"{task_id_list}"

#: apps/backend/subscription/errors.py:146
msgid "不存在运行中的任务，请确认"
msgstr "No running tasks, please confirm"

#: apps/backend/subscription/errors.py:147
#, python-brace-format
msgid "订阅[id:{subscription_id}]不存在运行中的任务，请确认"
msgstr ""
"Subscription [id:{subscription_id}] has no running tasks, please confirm"

#: apps/backend/subscription/handler.py:93
#, python-brace-format
msgid "订阅[{subscription_id}] 不存在关联的任务"
msgstr "Subscription [{subscription_id}] has no associated tasks"

#: apps/backend/subscription/steps/adapter.py:227
#, python-brace-format
msgid "插件包 [{name}-{version}] 不存在"
msgstr "Plugin package [{name}-{version}] does not exist"

#: apps/backend/subscription/steps/adapter.py:330
#, python-brace-format
msgid "订阅安装插件[{plugin_name}] 未选择支持 [{os_key}] 的插件包"
msgstr ""
"Subscription install plugin [{plugin_name}] has not selected a plugin "
"package that supports [{os_key}]"

#: apps/backend/subscription/steps/adapter.py:345
#, python-brace-format
msgid ""
"插件 [{name}] 不支持 系统:{os_type}-架构:{cpu_arch}-版本:{plugin_version}"
msgstr ""
"Plugin [{name}] does not support system:{os_type}-arch:{cpu_arch}-version:"
"{plugin_version}"

#: apps/backend/subscription/steps/adapter.py:354
#, python-brace-format
msgid ""
"插件 [{name}] 系统:{os_type}-架构:{cpu_arch}-版本:{plugin_version} 未启用"
msgstr ""
"Plugin [{name}] system:{os_type}-arch:{cpu_arch}-version:{plugin_version} is "
"not enabled"

#: apps/backend/subscription/steps/agent.py:228
#: apps/backend/subscription/steps/agent.py:249
#: apps/backend/subscription/steps/agent.py:277
#: apps/backend/subscription/steps/agent.py:351
msgid "查询Proxy状态"
msgstr "Query Proxy Status"

#: apps/backend/subscription/tasks.py:328
#, python-brace-format
msgid ""
"当前{category_alias}（{bk_obj_name} 级）已被优先级更高的"
"{suppressed_by_category_alias}【{suppressed_by_name}(ID: "
"{suppressed_by_id})】（{suppressed_by_obj_name} 级）抑制"
msgstr ""
"The current {category_alias} ({bk_obj_name} level) has been replaced by a "
"higher policy{suppressed_by_category_alias}[{suppressed_by_name}(ID: "
"{suppressed_by_id})] ({suppressed_by_obj_name} level) suppressed"

#: apps/backend/utils/ssh.py:191
msgid "认证方式错误或不支持，请确认。"
msgstr "The authentication method is wrong or not supported, please confirm."

#: apps/backend/utils/ssh.py:202
msgid "HostKey校验失败，请尝试删除 /root/.ssh/known_hosts 再重试"
msgstr ""
"HostKey verification failed, please try to delete /root/.ssh/known_hosts and "
"try again"

#: apps/backend/utils/ssh.py:213
msgid "登录认证失败，请确认账号，密码，密钥或IP是否正确。"
msgstr ""
"Login authentication failed, please confirm the account, password, key or IP "
"is correct."

#: apps/backend/utils/ssh.py:219
msgid "ssh登录，请确认IP是否正确或目标机器是否可被正常登录。"
msgstr ""
"ssh login, please confirm whether the IP is correct or whether the target "
"machine can be logged in normally."

#: apps/backend/utils/ssh.py:225
msgid ""
"ssh登录连接超时，请确认IP是否正确或ssh端口号是否正确或网络策略是否正确开通。"
msgstr ""
"The ssh login connection timed out, please confirm whether the IP is correct "
"or the ssh port number is correct or the network policy is correctly enabled."

#: apps/core/concurrent/controller.py:37
#, python-brace-format
msgid "解析并发控制配置失败，报错原因 -> {err}"
msgstr ""
"Failed to parse concurrency control configuration, error reason -> {err}"

#: apps/core/concurrent/serializers.py:20
msgid "是否全量执行"
msgstr "Whether full execution"

#: apps/core/concurrent/serializers.py:23
msgid "每批任务的执行数量限制"
msgstr "The limit on the number of tasks to execute per batch"

#: apps/core/concurrent/serializers.py:29
msgid "批次间是否并发执行"
msgstr "Whether batches are executed concurrently"

#: apps/core/concurrent/serializers.py:34
msgid "任务提交间隔"
msgstr "Task submission interval"

#: apps/core/encrypt/constants.py:25
msgid "公钥"
msgstr "Public key"

#: apps/core/encrypt/constants.py:25
msgid "私钥"
msgstr "Private key"

#: apps/core/encrypt/constants.py:33
msgid "默认RSA密钥"
msgstr "Default RSA key"

#: apps/core/encrypt/models.py:19 apps/core/encrypt/serializers.py:18
msgid "密钥名称"
msgstr "Key name"

#: apps/core/encrypt/models.py:20
msgid "密钥类型"
msgstr "Key Type"

#: apps/core/encrypt/models.py:21
msgid "密钥描述"
msgstr "Key description"

#: apps/core/encrypt/models.py:22
msgid "密钥内容"
msgstr "Key Content"

#: apps/core/encrypt/models.py:24 apps/core/files/models.py:27
#: apps/core/files/models.py:46 apps/node_man/models.py:301
#: apps/node_man/models.py:333 apps/node_man/models.py:1560
#: apps/node_man/models.py:1801 apps/node_man/models.py:2151
#: apps/node_man/models.py:2308
msgid "更新时间"
msgstr "Update time"

#: apps/core/encrypt/models.py:25 apps/core/files/models.py:28
#: apps/core/files/models.py:47 apps/node_man/models.py:332
#: apps/node_man/models.py:1559 apps/node_man/models.py:1585
#: apps/node_man/models.py:1662 apps/node_man/models.py:1802
#: apps/node_man/models.py:2111 apps/node_man/models.py:2152
#: apps/node_man/models.py:2288 apps/node_man/models.py:2309
#: apps/node_man/models.py:2325 apps/node_man/models.py:2368
#: pipeline/engine/models/core.py:882 pipeline/models.py:78
#: pipeline/models.py:275 pipeline/models.py:617
msgid "创建时间"
msgstr "Creation time"

#: apps/core/encrypt/models.py:28 apps/core/encrypt/models.py:29
msgid "RSA密钥"
msgstr "RSA key"

#: apps/core/encrypt/serializers.py:18
msgid "密钥名称列表"
msgstr "List of key names"

#: apps/core/files/base.py:109
msgid "未知文件源"
msgstr "Unknown file source"

#: apps/core/files/base.py:112
#, python-brace-format
msgid "{file_type_alias}-{file_source_alias}"
msgstr "{file_type_alias}-{file_source_alias}"

#: apps/core/files/base.py:120
#, python-brace-format
msgid "{file_type_alias}-{server_ips_str}"
msgstr "{file_type_alias}-{server_ips_str}"

#: apps/core/files/base.py:124
#, python-brace-format
msgid "未知文件源-{file_type}"
msgstr "Unknown file source - {file_type}"

#: apps/core/files/base.py:127
#, python-brace-format
msgid ""
"从 [{source_info_str}] 下发文件 [{file_list_str}] 到目标机器路径 "
"[{file_target_path}]"
msgstr ""
"Transfer file [{file_list_str}] from [{source_info_str}] to target machine "
"path [{file_target_path}]"

#: apps/core/files/base.py:216
#, python-brace-format
msgid "文件分发错误：err_msg -> {err_msg}"
msgstr "File distribution error: err_msg -> {err_msg}"

#: apps/core/files/constants.py:29
msgid "私有仓库"
msgstr "Private repository"

#: apps/core/files/constants.py:29
msgid "公共仓库"
msgstr "Public repository"

#: apps/core/files/constants.py:40
msgid "服务器文件"
msgstr "Server file"

#: apps/core/files/constants.py:40
msgid "第三方文件源文件"
msgstr "Third party file source file"

#: apps/core/files/constants.py:55
msgid "蓝鲸制品库"
msgstr "BKREPO"

#: apps/core/files/constants.py:55
msgid "本地文件系统"
msgstr "Local filesystem"

#: apps/core/files/exceptions.py:22
msgid "注册凭证失败"
msgstr "Failed to register credentials"

#: apps/core/files/exceptions.py:27
msgid "文件源类型错误"
msgstr "File source type error"

#: apps/core/files/exceptions.py:32
msgid "注册文件失败"
msgstr "Failed to register file"

#: apps/core/files/exceptions.py:37 apps/node_man/constants.py:698
msgid "文件传输失败"
msgstr "File transfer failed"

#: apps/core/files/file_source.py:25
#, python-brace-format
msgid "storage_type must be one of {choices}"
msgstr "storage_type must be one of {choices}"

#: apps/core/files/file_source.py:33
#, python-brace-format
msgid "节点管理[biz:{bk_biz_id}]{storage_type_alias}文件凭证"
msgstr "Node management [biz:{bk_biz_id}]{storage_type_alias} file credential"

#: apps/core/files/file_source.py:92
#, python-brace-format
msgid "注册凭证失败：bk_biz_id -> {bk_biz_id}, err_msg -> {err_msg}"
msgstr ""
"Failed to register credentials: bk_biz_id -> {bk_biz_id}, err_msg -> "
"{err_msg}"

#: apps/core/files/file_source.py:145
#, python-brace-format
msgid "节点管理[biz:{bk_biz_id}]{storage_type_alias}文件源"
msgstr "Node management [biz:{bk_biz_id}]{storage_type_alias} file source"

#: apps/core/files/file_source.py:200
#, python-brace-format
msgid "注册文件源失败：bk_biz_id -> {bk_biz_id}, err_msg -> {err_msg}"
msgstr ""
"Failed to register file source: bk_biz_id -> {bk_biz_id}, err_msg -> "
"{err_msg}"

#: apps/core/files/models.py:19 apps/node_man/models.py:310
#: apps/node_man/models.py:1556 apps/node_man/models.py:1795
#: apps/node_man/serializers/cmdb.py:39 apps/node_man/serializers/cmdb.py:44
#: apps/node_man/serializers/host.py:19 apps/node_man/serializers/host.py:37
#: apps/node_man/serializers/host.py:72 apps/node_man/serializers/host_v2.py:26
#: apps/node_man/serializers/job.py:30 apps/node_man/serializers/job.py:46
#: apps/node_man/serializers/job.py:146 apps/node_man/serializers/plugin.py:195
#: apps/node_man/serializers/plugin.py:228
#: apps/node_man/serializers/plugin_v2.py:173
#: apps/node_man/serializers/plugin_v2.py:241
#: apps/node_man/serializers/policy.py:88
#: apps/node_man/serializers/policy.py:112
msgid "业务ID"
msgstr "Business ID"

#: apps/core/files/models.py:20
msgid "文件源存储类型"
msgstr "File source storage type"

#: apps/core/files/models.py:22
msgid "凭证名称"
msgstr "Credential name"

#: apps/core/files/models.py:23
msgid "凭证类型"
msgstr "Credential type"

#: apps/core/files/models.py:24
msgid "凭证描述"
msgstr "Credential description"

#: apps/core/files/models.py:25 apps/core/files/models.py:38
msgid "凭证ID"
msgstr "Credential ID"

#: apps/core/files/models.py:31 apps/core/files/models.py:32
msgid "作业平台文件凭证"
msgstr "Job platform file credentials"

#: apps/core/files/models.py:39
msgid "文件源ID"
msgstr "File source ID"

#: apps/core/files/models.py:41 apps/core/files/models.py:44
msgid "文件源标识"
msgstr "File source ID"

#: apps/core/files/models.py:42
msgid "文件源别名"
msgstr "File source alias"

#: apps/core/files/models.py:43
msgid "文件源接入参数"
msgstr "File Source Access Parameters"

#: apps/core/files/models.py:62 apps/core/files/models.py:63
msgid "作业平台文件源"
msgstr "JOB file source"

#: apps/core/remote/conns/asyncssh_impl.py:72
#: apps/core/remote/conns/paramiko_impl.py:88
#, python-brace-format
msgid "连接超时：{err_msg}"
msgstr "Connection timed out: {err_msg}"

#: apps/core/remote/conns/asyncssh_impl.py:86
#, python-brace-format
msgid "命令执行超时：{err_msg}"
msgstr "Command execution timed out: {err_msg}"

#: apps/core/remote/conns/asyncssh_impl.py:93
#, python-brace-format
msgid "exit_status -> {exit_status}, stdout -> {stdout}, stderr -> {stderr}"
msgstr "exit_status -> {exit_status}, stdout -> {stdout}, stderr -> {stderr}"

#: apps/core/remote/conns/paramiko_impl.py:65
#, python-brace-format
msgid "密钥验证失败：{err_msg}"
msgstr "Key verification failed: {err_msg}"

#: apps/core/remote/exceptions.py:22
#, python-brace-format
msgid "命令执行失败: {err_msg}"
msgstr "Command execution failed: {err_msg}"

#: apps/core/remote/exceptions.py:23
msgid "命令执行失败"
msgstr "Command execution failed"

#: apps/core/remote/exceptions.py:28
#, python-brace-format
msgid "密钥交换失败: {err_msg}"
msgstr "Key exchange failed: {err_msg}"

#: apps/core/remote/exceptions.py:33
#, python-brace-format
msgid "认证失败，请检查认证信息是否有误：{err_msg}"
msgstr ""
"Authentication failed, please check the authentication information for "
"errors: {err_msg}"

#: apps/core/remote/exceptions.py:38
#, python-brace-format
msgid "连接丢失，远程系统未响应或登录超时：{err_msg}"
msgstr ""
"Connection lost, remote system not responding or login timed out: {err_msg}"

#: apps/core/remote/exceptions.py:43
#, python-brace-format
msgid "远程连接失败：{err_msg}"
msgstr "Remote connection failed: {err_msg}"

#: apps/core/remote/exceptions.py:48
msgid "超时"
msgstr "Timeout"

#: apps/core/remote/exceptions.py:53
#, python-brace-format
msgid "命令返回非零值：{err_msg}"
msgstr "Command returned non-zero value: {err_msg}"

#: apps/core/remote/exceptions.py:58
#, python-brace-format
msgid "会话异常：{err_msg}"
msgstr "Session exception: {err_msg}"

#: apps/core/remote/exceptions.py:63
#, python-brace-format
msgid "IO 异常：{err_msg}"
msgstr "IO exception: {err_msg}"

#: apps/core/remote/exceptions.py:68
#, python-brace-format
msgid "文件远程客户端异常：{err_msg}"
msgstr "File remote client exception: {err_msg}"

#: apps/exceptions.py:24
msgid "系统异常"
msgstr "System exception"

#: apps/exceptions.py:63 apps/exceptions.py:122
msgid "参数验证失败"
msgstr "Parameter validation failed"

#: apps/exceptions.py:68
msgid "远程服务请求结果异常"
msgstr "The remote service request result was abnormal"

#: apps/exceptions.py:73
msgid "组件调用异常"
msgstr "Component call exception"

#: apps/exceptions.py:78
msgid "业务不存在"
msgstr "Business does not exist"

#: apps/exceptions.py:83
msgid "语言不支持"
msgstr "Language not supported"

#: apps/exceptions.py:88
msgid "权限不足"
msgstr "Insufficient permissions"

#: apps/exceptions.py:94
msgid "服务不稳定，请检查组件健康状况"
msgstr "Service unstable, please check component health"

#: apps/exceptions.py:100
msgid "请升级blueapps至最新版本"
msgstr "Please upgrade blueapps to the latest version"

#: apps/exceptions.py:105
msgid "获取JWT信息异常"
msgstr "Error getting JWT information"

#: apps/exceptions.py:110
msgid "JWT校验失败"
msgstr "JWT validation failed"

#: apps/generic.py:181 apps/middlewares.py:173
#, python-brace-format
msgid "【APP 自定义异常】{message}, code={code}, args={args}"
msgstr "[APP custom exception]{message}, code={code}, args={args}"

#: apps/generic.py:199
msgid "系统错误，请联系管理员"
msgstr "System error, please contact administrator"

#: apps/iam/exceptions.py:43
#, python-brace-format
msgid "当前用户无 [{action_name}] 权限"
msgstr "Current user does not have permission for [{action_name}]"

#: apps/iam/handlers/actions.py:174
#, python-brace-format
msgid "动作ID不存在：{action_id}"
msgstr "Action ID does not exist: {action_id}"

#: apps/iam/handlers/permission.py:243
#, python-brace-format
msgid "获取系统信息错误：{message}"
msgstr "Error getting system information: {message}"

#: apps/iam/handlers/permission.py:257
msgid "配置平台"
msgstr "CMDB"

#: apps/iam/handlers/resources.py:177
#, python-brace-format
msgid "资源ID不存在：{resource_id}"
msgstr "Resource ID does not exist: {resource_id}"

#: apps/iam/serializers.py:17
msgid "资源类型"
msgstr "Resource type"

#: apps/iam/serializers.py:18
msgid "资源ID"
msgstr "Resource ID"

#: apps/iam/serializers.py:20
msgid "动作ID列表"
msgstr "Action ID List"

#: apps/iam/serializers.py:21
msgid "资源列表"
msgstr "Resource list"

#: apps/middlewares.py:162
msgid "S-mart应用仅支持正式环境部署！"
msgstr "S-mart application only supports production environment deployment!"

#: apps/node_man/constants.py:74 apps/node_man/handlers/cloud.py:108
#: apps/node_man/handlers/meta.py:349 apps/node_man/models.py:807
msgid "直连区域"
msgstr "Direct area"

#: apps/node_man/constants.py:155
msgid "Agent"
msgstr "Agent"

#: apps/node_man/constants.py:155
msgid "插件"
msgstr "Plugin"

#: apps/node_man/constants.py:155
msgid "Proxy"
msgstr "Proxy"

#: apps/node_man/constants.py:220
msgid "安装 Proxy"
msgstr "Install Proxy"

#: apps/node_man/constants.py:221
msgid "安装 Agent"
msgstr "Install Agent"

#: apps/node_man/constants.py:222
msgid "重启 Agent"
msgstr "Restart Agent"

#: apps/node_man/constants.py:223
msgid "重启 Proxy"
msgstr "Restart Proxy"

#: apps/node_man/constants.py:224
msgid "替换 Proxy"
msgstr "Replace Proxy"

#: apps/node_man/constants.py:225
msgid "重装 Proxy"
msgstr "Reinstall Proxy"

#: apps/node_man/constants.py:226
msgid "重装 Agent"
msgstr "Reinstall Agent"

#: apps/node_man/constants.py:227
msgid "升级 Proxy"
msgstr "Upgrade Proxy"

#: apps/node_man/constants.py:228
msgid "升级 Agent"
msgstr "Upgrade Agent"

#: apps/node_man/constants.py:229
msgid "移除 Agent"
msgstr "Remove Agent"

#: apps/node_man/constants.py:230
msgid "卸载 Agent"
msgstr "Uninstall Agent"

#: apps/node_man/constants.py:231
msgid "卸载 Proxy"
msgstr "Uninstall Proxy"

#: apps/node_man/constants.py:232
msgid "导入Proxy机器"
msgstr "Import proxy machine"

#: apps/node_man/constants.py:233
msgid "导入Agent机器"
msgstr "Import Agent Machine"

#: apps/node_man/constants.py:234
msgid "启动插件"
msgstr "Start plugin"

#: apps/node_man/constants.py:235
msgid "停止插件"
msgstr "Stop plugin"

#: apps/node_man/constants.py:236
msgid "重启插件"
msgstr "Restart plugin"

#: apps/node_man/constants.py:237
msgid "重载插件"
msgstr "Reload plugin"

#: apps/node_man/constants.py:238
msgid "托管插件"
msgstr "Supervised plugin"

#: apps/node_man/constants.py:239
msgid "取消托管插件"
msgstr "Unsupervised plugin"

#: apps/node_man/constants.py:240
msgid "安装插件"
msgstr "Install plugin"

#: apps/node_man/constants.py:241
msgid "停止插件并删除策略"
msgstr "Stop plugin and remove policy"

#: apps/node_man/constants.py:242 apps/node_man/constants.py:243
msgid "重载配置"
msgstr "Reload configuration"

#: apps/node_man/constants.py:244
msgid "打包插件"
msgstr "Package plugin"

#: apps/node_man/constants.py:245
msgid "下发插件配置"
msgstr "Push plugin config"

#: apps/node_man/constants.py:246
msgid "移除插件配置"
msgstr "Remove plugin configuration"

#: apps/node_man/constants.py:248
msgid "手动安装 Agent"
msgstr "Manually install the agent"

#: apps/node_man/constants.py:249
msgid "手动安装 Proxy"
msgstr "Manually install Proxy"

#: apps/node_man/constants.py:284
msgid "替换"
msgstr "Replace"

#: apps/node_man/constants.py:285
msgid "升级"
msgstr "Upgrade"

#: apps/node_man/constants.py:286
msgid "重装"
msgstr "Reinstall"

#: apps/node_man/constants.py:287
msgid "更新"
msgstr "Update"

#: apps/node_man/constants.py:288
msgid "移除"
msgstr "Remove"

#: apps/node_man/constants.py:289
msgid "卸载"
msgstr "Uninstall"

#: apps/node_man/constants.py:290
msgid "导入"
msgstr "Import"

#: apps/node_man/constants.py:291 apps/node_man/constants.py:459
msgid "启动"
msgstr "Start"

#: apps/node_man/constants.py:292 apps/node_man/constants.py:460
msgid "停止"
msgstr "Stop"

#: apps/node_man/constants.py:293 apps/node_man/constants.py:462
msgid "重载"
msgstr "Reload"

#: apps/node_man/constants.py:294 apps/node_man/constants.py:463
msgid "托管"
msgstr "Supervised"

#: apps/node_man/constants.py:295 apps/node_man/constants.py:464
msgid "取消托管"
msgstr "Unsupervised"

#: apps/node_man/constants.py:296
msgid "手动安装"
msgstr "Manual install"

#: apps/node_man/constants.py:297
msgid "打包"
msgstr "Packaging"

#: apps/node_man/constants.py:298 apps/node_man/models.py:502
#: apps/node_man/models.py:1787
msgid "调试"
msgstr "Debug"

#: apps/node_man/constants.py:299
msgid "卸载并删除"
msgstr "Uninstall and delete"

#: apps/node_man/constants.py:300
msgid "下发"
msgstr "Push"

#: apps/node_man/constants.py:301
msgid "忽略"
msgstr "Ignore"

#: apps/node_man/constants.py:303
msgid "下发配置"
msgstr "Push config"

#: apps/node_man/constants.py:304
msgid "策略管控"
msgstr "Policy control"

#: apps/node_man/constants.py:329
msgid "云区域查看"
msgstr "Cloud area view"

#: apps/node_man/constants.py:330
msgid "云区域编辑"
msgstr "Cloud area edit"

#: apps/node_man/constants.py:331
msgid "云区域删除"
msgstr "Cloud area delete"

#: apps/node_man/constants.py:332
msgid "云区域创建"
msgstr "Cloud area created"

#: apps/node_man/constants.py:333
msgid "接入点编辑"
msgstr "Access Point Edit"

#: apps/node_man/constants.py:334
msgid "接入点删除"
msgstr "Access point deleted"

#: apps/node_man/constants.py:335
msgid "接入点创建"
msgstr "Access Point Creation"

#: apps/node_man/constants.py:336
msgid "接入点查看"
msgstr "Access Point View"

#: apps/node_man/constants.py:337
msgid "任务配置"
msgstr "Task configuration"

#: apps/node_man/constants.py:338
msgid "任务历史查看"
msgstr "Task history view"

#: apps/node_man/constants.py:339
msgid "agent查询"
msgstr "Agent query"

#: apps/node_man/constants.py:340
msgid "agent操作"
msgstr "Agent operation"

#: apps/node_man/constants.py:341
msgid "proxy操作"
msgstr "Proxy operation"

#: apps/node_man/constants.py:342
msgid "插件实例查询"
msgstr "Plugin instance query"

#: apps/node_man/constants.py:343
msgid "插件实例操作"
msgstr "Plugin instance action"

#: apps/node_man/constants.py:344
msgid "插件包导入"
msgstr "Plugin package import"

#: apps/node_man/constants.py:345
msgid "插件包操作"
msgstr "Plugin package action"

#: apps/node_man/constants.py:346
msgid "策略创建"
msgstr "Policy creation"

#: apps/node_man/constants.py:347
msgid "策略查看"
msgstr "Policy view"

#: apps/node_man/constants.py:348
msgid "策略编辑"
msgstr "Policy edit"

#: apps/node_man/constants.py:349
msgid "策略目标选择"
msgstr "Policy target selection"

#: apps/node_man/constants.py:375 apps/node_man/constants.py:686
#: apps/node_man/constants.py:722 apps/node_man/tools/job.py:55
msgid "等待执行"
msgstr "Waiting for execution"

#: apps/node_man/constants.py:379
msgid "部分失败"
msgstr "Partial failure"

#: apps/node_man/constants.py:380
msgid "已终止"
msgstr "Terminated"

#: apps/node_man/constants.py:381
msgid "已移除"
msgstr "Removed"

#: apps/node_man/constants.py:382
msgid "被过滤的"
msgstr "Filtered"

#: apps/node_man/constants.py:383
msgid "已忽略"
msgstr "Ignored"

#: apps/node_man/constants.py:400 apps/node_man/constants.py:413
msgid "未知"
msgstr "Unknown"

#: apps/node_man/constants.py:400 apps/node_man/constants.py:416
#: apps/node_man/handlers/meta.py:320
msgid "正常"
msgstr "Running"

#: apps/node_man/constants.py:400 apps/node_man/constants.py:417
#: apps/node_man/handlers/meta.py:321
msgid "异常"
msgstr "Terminal"

#: apps/node_man/constants.py:400 apps/node_man/constants.py:414
msgid "未安装"
msgstr "Uninstall"

#: apps/node_man/constants.py:415 apps/node_man/handlers/meta.py:322
msgid "未注册"
msgstr "Unregistered"

#: apps/node_man/constants.py:418
msgid "手动停止"
msgstr "Manual stop"

#: apps/node_man/constants.py:419
msgid "成功"
msgstr "Success"

#: apps/node_man/constants.py:420
msgid "失败"
msgstr "Failed"

#: apps/node_man/constants.py:421
msgid "队列中"
msgstr "In queue"

#: apps/node_man/constants.py:451 apps/node_man/constants.py:456
msgid "官方插件"
msgstr "official plugin"

#: apps/node_man/constants.py:452 apps/node_man/constants.py:456
msgid "第三方插件"
msgstr "third-party plugin"

#: apps/node_man/constants.py:453 apps/node_man/constants.py:456
msgid "脚本插件"
msgstr "script plugin"

#: apps/node_man/constants.py:494 apps/node_man/constants.py:498
msgid "整包部署"
msgstr "full package deployment"

#: apps/node_man/constants.py:495 apps/node_man/constants.py:498
msgid "功能部署"
msgstr "feature deployment"

#: apps/node_man/constants.py:496 apps/node_man/constants.py:498
msgid "Agent自动部署"
msgstr "Agent auto-deployment"

#: apps/node_man/constants.py:504
msgid "发布（上线）插件包"
msgstr "Publish (release) plugin package"

#: apps/node_man/constants.py:505
msgid "下线插件包"
msgstr "Offline plugin package"

#: apps/node_man/constants.py:506
msgid "启用插件包"
msgstr "Enable plugin package"

#: apps/node_man/constants.py:507
msgid "停用插件包"
msgstr "Deactivate plugin package"

#: apps/node_man/constants.py:513
msgid "启用插件"
msgstr "Enable plugin"

#: apps/node_man/constants.py:513
msgid "停用插件"
msgstr "Deactivate plugin"

#: apps/node_man/constants.py:682 apps/node_man/constants.py:719
msgid "作业平台API返回中不存在此IP的日志，请联系管理员排查问题"
msgstr ""
"The log of this IP does not exist in the API return of the job platform, "
"please contact the administrator to troubleshoot the problem"

#: apps/node_man/constants.py:683 apps/node_man/constants.py:697
#: apps/node_man/constants.py:701 apps/node_man/constants.py:720
msgid "Agent异常"
msgstr "Agent exception"

#: apps/node_man/constants.py:684
msgid "上次已成功"
msgstr "Last time was successful"

#: apps/node_man/constants.py:689 apps/node_man/models.py:1422
msgid "任务失败"
msgstr "Task failed"

#: apps/node_man/constants.py:690 apps/node_man/constants.py:726
msgid "任务下发失败"
msgstr "Task delivery failed"

#: apps/node_man/constants.py:691
msgid "任务超时"
msgstr "Task timed out"

#: apps/node_man/constants.py:692
msgid "任务日志错误"
msgstr "Task log error"

#: apps/node_man/constants.py:693
msgid "脚本执行失败"
msgstr "Script execution failed"

#: apps/node_man/constants.py:694
msgid "脚本执行超时"
msgstr "Script execution timed out"

#: apps/node_man/constants.py:695
msgid "脚本执行被终止"
msgstr "Script execution terminated"

#: apps/node_man/constants.py:696
msgid "脚本返回码非零"
msgstr "Script return code is non-zero"

#: apps/node_man/constants.py:699
msgid "源文件不存在"
msgstr "Source file does not exist"

#: apps/node_man/constants.py:700
msgid "文件任务超时"
msgstr "File task timed out"

#: apps/node_man/constants.py:702
msgid "用户名不存在"
msgstr "Username does not exist"

#: apps/node_man/constants.py:703
msgid "用户密码错误"
msgstr "Incorrect user password"

#: apps/node_man/constants.py:704
msgid "文件获取失败"
msgstr "Failed to get file"

#: apps/node_man/constants.py:705
msgid "文件超出限制"
msgstr "File exceeds limit"

#: apps/node_man/constants.py:706
msgid "文件传输错误"
msgstr "File transfer error"

#: apps/node_man/constants.py:707
msgid "任务执行出错"
msgstr "Error executing task"

#: apps/node_man/constants.py:708
msgid "任务被强制终止"
msgstr "Task force terminated"

#: apps/node_man/constants.py:727
msgid "任务强制终止成功"
msgstr "Task force terminated successfully"

#: apps/node_man/constants.py:728
msgid "任务强制终止失败"
msgstr "Task force termination failed"

#: apps/node_man/constants.py:740
msgid "是，参数将会在执行详情页面上隐藏"
msgstr "Yes, the parameter will be hidden on the execution details page"

#: apps/node_man/constants.py:740
msgid "否，默认值"
msgstr "no, default"

#: apps/node_man/constants.py:751 apps/node_man/constants.py:808
msgid "业务"
msgstr "Business"

#: apps/node_man/constants.py:751
msgid "业务集"
msgstr "Business set"

#: apps/node_man/constants.py:766
msgid "shell"
msgstr "shell"

#: apps/node_man/constants.py:767
msgid "bat"
msgstr "bat"

#: apps/node_man/constants.py:768
msgid "perl"
msgstr "perl"

#: apps/node_man/constants.py:769
msgid "python"
msgstr "python"

#: apps/node_man/constants.py:770
msgid "power_shell"
msgstr "power_shell"

#: apps/node_man/constants.py:788
msgid "启动进程"
msgstr "Start process"

#: apps/node_man/constants.py:789
msgid "停止进程"
msgstr "Stop process"

#: apps/node_man/constants.py:790
msgid "查询进程"
msgstr "Query process"

#: apps/node_man/constants.py:791
msgid "托管进程"
msgstr "Supervised process"

#: apps/node_man/constants.py:792
msgid "取消托管进程"
msgstr "Unsupervised process"

#: apps/node_man/constants.py:793
msgid "重启进程"
msgstr "Restart process"

#: apps/node_man/constants.py:794
msgid "重载进程"
msgstr "Reload process"

#: apps/node_man/constants.py:808
msgid "集群"
msgstr "Cluster"

#: apps/node_man/constants.py:808
msgid "模块"
msgstr "Module"

#: apps/node_man/constants.py:808 apps/node_man/models.py:1764
msgid "主机"
msgstr "Host"

#: apps/node_man/constants.py:808
msgid "自定义"
msgstr "Custom"

#: apps/node_man/constants.py:820
msgid "检测 BT 分发策略（下发Py36包）"
msgstr "Detect BT distribution strategy (distribute Py36 package)"

#: apps/node_man/constants.py:844
msgid "下发安装工具"
msgstr "transfer installation tool"

#: apps/node_man/exceptions.py:21
msgid "云区域不存在"
msgstr "Cloud area does not exist"

#: apps/node_man/exceptions.py:26
msgid "云区域业务已装Agent, 不可删除"
msgstr ""
"Agent has been installed in the cloud area business and cannot be deleted"

#: apps/node_man/exceptions.py:31
msgid "该云区域已经存在主机, 不可删除"
msgstr "The host already exists in this cloud area and cannot be deleted"

#: apps/node_man/exceptions.py:36
msgid "PAGENT只能安装在非直连区域"
msgstr "PAGENT can only be installed in non-directly connected areas"

#: apps/node_man/exceptions.py:41
msgid "AGENT只能安装在直连区域"
msgstr "AGENT can only be installed in directly connected areas"

#: apps/node_man/exceptions.py:46
msgid "直连区域不可安装Proxy"
msgstr "A proxy cannot be installed in a directly connected area"

#: apps/node_man/exceptions.py:51
msgid "该接入点不存在."
msgstr "The access point does not exist."

#: apps/node_man/exceptions.py:52
#, python-brace-format
msgid "该接入点[{ap_id}]不存在"
msgstr "The access point [{ap_id}] does not exist"

#: apps/node_man/exceptions.py:57
msgid "该Ip已被占用."
msgstr "This IP is already occupied."

#: apps/node_man/exceptions.py:62
msgid "不存在某Host ID."
msgstr "A Host ID does not exist."

#: apps/node_man/exceptions.py:67
msgid "认证信息校验不通过"
msgstr "Authentication information verification failed"

#: apps/node_man/exceptions.py:77
msgid "没有可用的Proxy"
msgstr "No Proxy available"

#: apps/node_man/exceptions.py:82
msgid "不存在某云区域的权限."
msgstr "There is no permission for a cloud area."

#: apps/node_man/exceptions.py:87
msgid "该主机未选择接入点，请尝试重装主机自动选择接入点"
msgstr ""
"The host did not select an access point, please try reinstalling the host to "
"automatically select an access point"

#: apps/node_man/exceptions.py:92
msgid "您不是超级用户无法访问此页面."
msgstr "You are not a superuser and cannot access this page."

#: apps/node_man/exceptions.py:97
msgid "所有ip均被过滤"
msgstr "all ips are filtered"

#: apps/node_man/exceptions.py:102
msgid "该接入点正在被使用"
msgstr "This access point is in use"

#: apps/node_man/exceptions.py:107
msgid "该IP正在运行其他任务"
msgstr "This IP is running another task"

#: apps/node_man/exceptions.py:112
msgid "该主机没有操作系统参数"
msgstr "The host has no operating system parameters"

#: apps/node_man/exceptions.py:117
msgid "您没有该主机的权限"
msgstr "You do not have permission to this host"

#: apps/node_man/exceptions.py:122
msgid "主机不存在"
msgstr "host does not exist"

#: apps/node_man/exceptions.py:127
msgid "不允许混合操作【手动安装】和【自动安装】的主机"
msgstr ""
"Mixing [manually installed] and [automatically installed] hosts is not "
"allowed"

#: apps/node_man/exceptions.py:132
msgid "配置Proxy策略失败"
msgstr "Failed to configure proxy policy"

#: apps/node_man/exceptions.py:137
msgid "权限中心请求失败"
msgstr "IAM request failed"

#: apps/node_man/exceptions.py:143
msgid "获取全局配置失败"
msgstr "Failed to get global configuration"

#: apps/node_man/exceptions.py:148
msgid "接入点名称重复"
msgstr "Duplicate access point name"

#: apps/node_man/exceptions.py:152
msgid "插件不存在"
msgstr "plugin does not exist"

#: apps/node_man/exceptions.py:157
msgid "缓存过期，正在重新加载..."
msgstr "Cache expired, reloading..."

#: apps/node_man/exceptions.py:162
msgid "策略不存在"
msgstr "Policy does not exist"

#: apps/node_man/exceptions.py:167
msgid "插件配置模板不存在"
msgstr "Plugin configuration template does not exist"

#: apps/node_man/exceptions.py:172
msgid "创建插件记录失败"
msgstr "Failed to create plugin record"

#: apps/node_man/exceptions.py:177
msgid "接入点不支持此机器的操作系统."
msgstr "The access point does not support this machine's operating system."

#: apps/node_man/exceptions.py:178
#, python-brace-format
msgid "接入点不[{ap_id}]支持此机器的操作系统[{os_type}]"
msgstr "Access point [{ap_id}] does not support this machine's OS [{os_type}]"

#: apps/node_man/exceptions.py:183
#, python-brace-format
msgid "策略 -> {policy_id}「{name}」正在执行"
msgstr "policy->{policy_id}{name} is running"

#: apps/node_man/exceptions.py:189
msgid "主机的安装通道不存在，请重新选择"
msgstr ""
"The installation channel for the host does not exist, please select again"

#: apps/node_man/exceptions.py:193
msgid "插件上传失败"
msgstr "Plugin upload failed"

#: apps/node_man/exceptions.py:194
#, python-brace-format
msgid "插件上传失败: plugin_name -> {plugin_name}, error -> {error}"
msgstr ""
"Failed to upload plugin: plugin_name -> {plugin_name}, error -> {error}"

#: apps/node_man/exceptions.py:199
msgid "访问地址不可达"
msgstr "Access address unreachable"

#: apps/node_man/exceptions.py:204
msgid "插件配置未变更，无需执行"
msgstr "Plugin configuration has not changed, no need to execute"

#: apps/node_man/handlers/cloud.py:46 apps/node_man/handlers/permission.py:109
#, python-brace-format
msgid "不存在ID为: {bk_cloud_id} 的云区域"
msgstr "No cloud area with id: {bk_cloud_id}"

#: apps/node_man/handlers/cloud.py:50 apps/node_man/handlers/cloud.py:137
msgid "自动选择接入点"
msgstr "Auto select access point"

#: apps/node_man/handlers/cloud.py:197
msgid "云区域不可名为直连区域"
msgstr "The cloud area cannot be named as a directly connected area"

#: apps/node_man/handlers/cloud.py:201 apps/node_man/handlers/cloud.py:229
msgid "云区域名称不可重复"
msgstr "cloud area name cannot be repeated"

#: apps/node_man/handlers/cloud.py:218 apps/node_man/handlers/policy.py:536
#: apps/node_man/views/ap.py:247
msgid "权限中心创建关联权限失败: {}"
msgstr "Permission Center failed to create associated permissions: {}"

#: apps/node_man/handlers/cloud.py:247
msgid "该区域已存在主机"
msgstr "Host already exists in this area"

#: apps/node_man/handlers/cmdb.py:389
msgid "在CMDB中，还有主机关联到当前云区域下，无法删除"
msgstr ""
"In the CMDB, there are hosts associated with the current cloud area and "
"cannot be deleted"

#: apps/node_man/handlers/healthz/saas_healthz.py:110
#, python-format
msgid "第三方组件%s异常"
msgstr "The third-party component %s is abnormal"

#: apps/node_man/handlers/healthz/saas_healthz.py:110
msgid "请检查esb及对应组件服务是否正常"
msgstr ""
"Please check whether the esb and corresponding component services are normal"

#: apps/node_man/handlers/healthz/saas_healthz.py:114
#: apps/node_man/serializers/job.py:28 pipeline/engine/models/core.py:876
msgid "状态"
msgstr "status"

#: apps/node_man/handlers/healthz/saas_healthz.py:177
#: apps/node_man/handlers/healthz/saas_healthz.py:248
#: apps/node_man/handlers/healthz/saas_healthz.py:319
msgid "{} 不存在对应的测试用例"
msgstr "{} does not have a corresponding test case"

#: apps/node_man/handlers/healthz/saas_healthz.py:201
#: apps/node_man/handlers/healthz/saas_healthz.py:272
#: apps/node_man/handlers/healthz/saas_healthz.py:343
#, python-brace-format
msgid "cc中不存在{method_name}"
msgstr "{method_name} does not exist in cc"

#: apps/node_man/handlers/healthz/saas_healthz.py:219
#: apps/node_man/handlers/healthz/saas_healthz.py:290
#: apps/node_man/handlers/healthz/saas_healthz.py:368
#: apps/node_man/handlers/healthz/saas_healthz.py:490
msgid "接口不具有依赖关系"
msgstr "interface has no dependencies"

#: apps/node_man/handlers/healthz/saas_healthz.py:222
#: apps/node_man/handlers/healthz/saas_healthz.py:293
#: apps/node_man/handlers/healthz/saas_healthz.py:371
msgid "cc中没有此接口"
msgstr "no such interface in cc"

#: apps/node_man/handlers/healthz/saas_healthz.py:414
msgid "job不存在对应的测试用例"
msgstr "job does not have a corresponding test case"

#: apps/node_man/handlers/healthz/saas_healthz.py:417
msgid "job 中不存在接口 fast_execute_script 的相关参数"
msgstr ""
"The related parameters of the interface fast_execute_script do not exist in "
"the job"

#: apps/node_man/handlers/healthz/saas_healthz.py:458
#, python-brace-format
msgid "job中不存在{method_name}"
msgstr "{method_name} does not exist in job"

#: apps/node_man/handlers/healthz/saas_healthz.py:469
#, python-format
msgid "查找业务下主机失败,主机查询结果:%s"
msgstr "Failed to find the host under the service, host query result: %s"

#: apps/node_man/handlers/healthz/saas_healthz.py:493
msgid "job中没有此接口"
msgstr "no such interface in job"

#: apps/node_man/handlers/host.py:626 apps/node_man/handlers/permission.py:410
#, python-brace-format
msgid "Host ID:{bk_host_id} 不存在"
msgstr "Host ID: {bk_host_id} does not exist"

#: apps/node_man/handlers/host.py:634
#, python-brace-format
msgid "外网IP：{outer_ip} 已被占用"
msgstr "External IP: {outer_ip} is occupied"

#: apps/node_man/handlers/host.py:639
#, python-brace-format
msgid "登录IP：{login_ip} 已被占用"
msgstr "Login IP: {login_ip} is occupied"

#: apps/node_man/handlers/host.py:647
#, python-brace-format
msgid "Ap_id:{ap_id} 不存在于数据库."
msgstr "Ap_id:{ap_id} does not exist in the database."

#: apps/node_man/handlers/host.py:743
#, python-brace-format
msgid "主机 {diff} 不存在"
msgstr "host {diff} does not exist"

#: apps/node_man/handlers/host.py:744
#, python-brace-format
msgid "您没有移除主机 {ips} 的权限"
msgstr "You do not have permission to remove host {ips}"

#: apps/node_man/handlers/job.py:48 apps/node_man/handlers/permission.py:390
#, python-brace-format
msgid "不存在ID为{job_id}的任务"
msgstr "No job with id {job_id} exists"

#: apps/node_man/handlers/job.py:829
msgid "没有需要变更的实例"
msgstr "No instance to change"

#: apps/node_man/handlers/meta.py:155
#, fuzzy
#| msgid "手动安装"
msgid "手动"
msgstr "Manual"

#: apps/node_man/handlers/meta.py:155
msgid "远程"
msgstr "Remote"

#: apps/node_man/handlers/meta.py:167 apps/node_man/handlers/meta.py:288
#: apps/node_man/handlers/meta.py:353 apps/node_man/models.py:319
#: apps/node_man/models.py:322 apps/node_man/models.py:1589
#: apps/node_man/serializers/job.py:55
msgid "操作系统"
msgstr "Operating system"

#: apps/node_man/handlers/meta.py:167
msgid "其它"
msgstr "other"

#: apps/node_man/handlers/meta.py:168 apps/node_man/handlers/meta.py:302
#: apps/node_man/handlers/meta.py:367
msgid "Agent状态"
msgstr "Agent Status"

#: apps/node_man/handlers/meta.py:169
msgid "安装方式"
msgstr "Install method"

#: apps/node_man/handlers/meta.py:170 apps/node_man/handlers/meta.py:295
#: apps/node_man/handlers/meta.py:419 apps/node_man/serializers/host.py:22
#: apps/node_man/serializers/host.py:73 apps/node_man/serializers/job.py:147
#: apps/node_man/serializers/plugin.py:199
#: apps/node_man/serializers/plugin.py:230
msgid "Agent版本"
msgstr "Agent version"

#: apps/node_man/handlers/meta.py:171 apps/node_man/handlers/meta.py:278
#: apps/node_man/handlers/meta.py:351
msgid "云区域"
msgstr "Cloud area"

#: apps/node_man/handlers/meta.py:172 apps/node_man/models.py:328
msgid "安装通道"
msgstr "Install channel"

#: apps/node_man/handlers/meta.py:173
msgid "IP"
msgstr "IP"

#: apps/node_man/handlers/meta.py:250 apps/node_man/models.py:2144
#: apps/node_man/serializers/debug.py:21 apps/node_man/serializers/job.py:27
msgid "任务ID"
msgstr "Task ID"

#: apps/node_man/handlers/meta.py:251 apps/node_man/serializers/job.py:29
#: pipeline/models.py:618
msgid "执行者"
msgstr "Executor"

#: apps/node_man/handlers/meta.py:252
msgid "执行状态"
msgstr "Task status"

#: apps/node_man/handlers/meta.py:253 apps/node_man/serializers/job.py:112
#: apps/node_man/serializers/job.py:145 apps/node_man/serializers/plugin.py:227
#: apps/node_man/serializers/plugin_v2.py:199
#: apps/node_man/serializers/policy.py:126
msgid "任务类型"
msgstr "Task type"

#: apps/node_man/handlers/meta.py:254 apps/node_man/serializers/iam.py:19
#: apps/node_man/serializers/job.py:117 apps/node_man/serializers/policy.py:142
msgid "操作类型"
msgstr "Operation type"

#: apps/node_man/handlers/meta.py:255
msgid "策略名称"
msgstr "Policy name"

#: apps/node_man/handlers/meta.py:316 apps/node_man/handlers/meta.py:434
msgid "无版本"
msgstr "No version"

#: apps/node_man/handlers/meta.py:318 apps/node_man/handlers/meta.py:440
msgid "{}状态"
msgstr "{}status"

#: apps/node_man/handlers/meta.py:378 apps/node_man/models.py:324
#: apps/node_man/models.py:1797 apps/node_man/serializers/job.py:118
#: apps/node_man/serializers/job.py:154 apps/node_man/serializers/plugin.py:241
msgid "节点类型"
msgstr "Node type"

#: apps/node_man/handlers/meta.py:432
#, python-brace-format
msgid "{name}版本"
msgstr ""

#: apps/node_man/handlers/permission.py:38
#: apps/node_man/handlers/permission.py:89
msgid "您没有权限执行操作"
msgstr "You do not have permission to perform the operation"

#: apps/node_man/handlers/permission.py:52
msgid "您没有该接入点的权限"
msgstr "You do not have permission to this access point"

#: apps/node_man/handlers/permission.py:74
msgid "您没有创建接入点的权限"
msgstr "You do not have permission to create an access point"

#: apps/node_man/handlers/permission.py:75
msgid "您没有编辑该接入点的权限"
msgstr "You do not have permission to edit this access point"

#: apps/node_man/handlers/permission.py:76
msgid "您没有删除该接入点的权限"
msgstr "You do not have permission to delete this access point"

#: apps/node_man/handlers/permission.py:77
msgid "您没有查看该接入点详情的权限"
msgstr "You do not have permission to view details of this access point"

#: apps/node_man/handlers/permission.py:140
msgid "您没有创建云区域的权限"
msgstr "You do not have permission to create cloud areas"

#: apps/node_man/handlers/permission.py:141
msgid "您没有编辑该云区域的权限"
msgstr "You do not have permission to edit this cloud area"

#: apps/node_man/handlers/permission.py:142
msgid "您没有删除该云区域的权限"
msgstr "You do not have permission to delete this cloud area"

#: apps/node_man/handlers/permission.py:143
msgid "您没有查看该云区域详情的权限"
msgstr "You do not have permission to view details for this cloud area"

#: apps/node_man/handlers/permission.py:155
msgid "您没有权限执行操作，请申请云区域编辑权限"
msgstr ""
"You do not have permission to perform the operation, please apply for cloud "
"area editing permission"

#: apps/node_man/handlers/permission.py:181
msgid "您没有查询Debug接口的权限"
msgstr "You do not have permission to query the Debug interface"

#: apps/node_man/handlers/permission.py:197
#: apps/node_man/handlers/permission.py:258
msgid "您没有该操作的权限"
msgstr "You do not have permission for this operation"

#: apps/node_man/handlers/permission.py:323
msgid "【资源池】业务暂不支持从权限中心申请，请在admin中将用户添加为superuser"
msgstr ""
"[Resource pool] service does not currently support application from the "
"permission center, please add the user as superuser in admin"

#: apps/node_man/handlers/plugin_v2.py:206
#, python-brace-format
msgid "插件配置模板{ids}不存在或已下线"
msgstr "Plugin configuration template {ids} does not exist or is offline"

#: apps/node_man/handlers/policy.py:81 apps/node_man/serializers/policy.py:149
#: apps/node_man/tools/policy.py:38
#, python-brace-format
msgid "不存在ID为: {id} 的策略"
msgstr "Policy with id: {id} does not exist"

#: apps/node_man/handlers/policy.py:654
#, python-brace-format
msgid ""
"{plugin_name}: {plugin_description} - V{version} \n"
" 该版本支持{plugin_scenario}"
msgstr ""
"{plugin_name}: {plugin_description} - V{version} \n"
"This version supports {plugin_scenario}"

#: apps/node_man/handlers/validator.py:226
msgid "修改了认证类型为秘钥认证，但是没有输入认证凭证"
msgstr ""
"Modified authentication type to key authentication, but no authentication "
"credentials have been entered"

#: apps/node_man/handlers/validator.py:230
msgid "修改了认证类型为密码认证，但是没有输入密码"
msgstr ""
"Modified authentication type to password authentication, but no password was "
"entered"

#: apps/node_man/handlers/validator.py:243
msgid "认证信息已过期，请输入相关认证信息"
msgstr ""
"The authentication information has expired, please enter the relevant "
"authentication information"

#: apps/node_man/handlers/validator.py:315
#, python-brace-format
msgid "主机(IP:{ip}) 没有操作系统, 请【重装】并补全相关信息"
msgstr ""
"The host (IP:{ip}) has no operating system, please [reinstall] and complete "
"the relevant information"

#: apps/node_man/handlers/validator.py:319
#, python-brace-format
msgid "云区域(ID:{bk_cloud_id}) 不存在"
msgstr "The cloud area (ID:{bk_cloud_id}) does not exist"

#: apps/node_man/handlers/validator.py:323
#, python-brace-format
msgid "{cloud_name} 是直连区域，不可以安装PROXY"
msgstr "{cloud_name} is a directly connected area, PROXY cannot be installed"

#: apps/node_man/handlers/validator.py:332
msgid "该云区域下不存在代理"
msgstr "No agent exists in this cloud area"

#: apps/node_man/handlers/validator.py:340
msgid "直连区域必须选择接入点"
msgstr "Access point must be selected for direct area"

#: apps/node_man/handlers/validator.py:344
#, python-brace-format
msgid "接入点(id:{ap_id})不存在"
msgstr "Access point (id: {ap_id}) does not exist"

#: apps/node_man/handlers/validator.py:355
#, python-brace-format
msgid ""
"\n"
"                该主机内网IP已存在于所选云区域：{cloud_name} 下,\n"
"                业务：{bk_biz_id},\n"
"                节点类型：{node_type}\n"
"                "
msgstr ""
"\n"
"                The host's intranet IP already exists in the selected cloud "
"area: {cloud_name},\n"
"                Business: {bk_biz_id},\n"
"                Node type: {node_type}\n"
"                "

#: apps/node_man/handlers/validator.py:375
#, python-brace-format
msgid "尚未被安装，无法执行 {op_type} 操作"
msgstr "Not installed, cannot perform {op_type} operation"

#: apps/node_man/handlers/validator.py:383
#, python-brace-format
msgid "Host ID 不正确，无法执行 {op_type} 操作"
msgstr "Incorrect Host ID for {op_type} operation"

#: apps/node_man/handlers/validator.py:394
msgid "正在执行其他任务，无法执行新任务"
msgstr "Another task is being executed, new task cannot be executed"

#: apps/node_man/handlers/validator.py:403
#, python-brace-format
msgid ""
"节点类型不正确，该主机是 {host_node_type}, 而请求的操作类型是 {node_type}"
msgstr ""
"Incorrect node type, the host is {host_node_type}, and the requested "
"operation type is {node_type}"

#: apps/node_man/handlers/validator.py:432
#, python-brace-format
msgid "主机(IP:{inner_ip}) 没有操作系统, 请【重装】并补全相关信息"
msgstr ""
"The host (IP:{inner_ip}) has no operating system, please [reinstall] and "
"complete the relevant information"

#: apps/node_man/handlers/validator.py:436
msgid "不允许操作【云区域不存在】的主机, 云区域id: {}"
msgstr ""
"Not allowed to operate the host of [cloud area does not exist], cloud area "
"id: {}"

#: apps/node_man/models.py:102
msgid "键"
msgstr "Key"

#: apps/node_man/models.py:103
msgid "值"
msgstr "Value"

#: apps/node_man/models.py:139 apps/node_man/models.py:140
msgid "配置表（GlobalSettings）"
msgstr "Configuration Table (GlobalSettings)"

#: apps/node_man/models.py:158
#, python-brace-format
msgid "后台获取密钥失败：{e}"
msgstr "Failed to get key in background: {e}"

#: apps/node_man/models.py:169
#, python-brace-format
msgid "SaaS获取密钥失败：{e}"
msgstr "SaaS failed to get key: {e}"

#: apps/node_man/models.py:291 apps/node_man/models.py:506
#: apps/node_man/models.py:866 apps/node_man/serializers/debug.py:25
#: apps/node_man/serializers/host.py:20 apps/node_man/serializers/host.py:41
#: apps/node_man/serializers/host.py:70 apps/node_man/serializers/host_v2.py:28
#: apps/node_man/serializers/job.py:48 apps/node_man/serializers/job.py:149
#: apps/node_man/serializers/job.py:213 apps/node_man/serializers/plugin.py:196
#: apps/node_man/serializers/plugin.py:236
#: apps/node_man/serializers/plugin_v2.py:174
#: apps/node_man/serializers/policy.py:137
msgid "主机ID"
msgstr "Host ID"

#: apps/node_man/models.py:293 apps/node_man/serializers/host.py:50
#: apps/node_man/serializers/job.py:56
msgid "认证类型"
msgstr "Auth type"

#: apps/node_man/models.py:295
msgid "账户名"
msgstr "Account name"

#: apps/node_man/models.py:296 apps/node_man/models.py:628
#: apps/node_man/serializers/host.py:51 apps/node_man/serializers/job.py:58
msgid "密码"
msgstr "Password"

#: apps/node_man/models.py:297 apps/node_man/serializers/job.py:59
msgid "端口"
msgstr "Port"

#: apps/node_man/models.py:298 apps/node_man/serializers/job.py:60
msgid "密钥"
msgstr "Key"

#: apps/node_man/models.py:299
msgid "额外认证资料"
msgstr "Additional credentials"

#: apps/node_man/models.py:300
msgid "保留天数"
msgstr "Retention days"

#: apps/node_man/models.py:304 apps/node_man/models.py:305
msgid "临时认证数据（IdentityData）"
msgstr "Temporary IdentityData"

#: apps/node_man/models.py:309
msgid "CMDB主机ID"
msgstr "CMDB host ID"

#: apps/node_man/models.py:311 apps/node_man/models.py:827
#: apps/node_man/serializers/host.py:21 apps/node_man/serializers/host.py:33
#: apps/node_man/serializers/host.py:42
#: apps/node_man/serializers/install_channel.py:20
#: apps/node_man/serializers/job.py:47 apps/node_man/serializers/plugin.py:197
#: apps/node_man/serializers/plugin.py:229
msgid "云区域ID"
msgstr "Cloud area ID"

#: apps/node_man/models.py:313 apps/node_man/serializers/ap.py:68
#: apps/node_man/serializers/host.py:43 apps/node_man/serializers/job.py:51
msgid "内网IP"
msgstr "Intranet IP"

#: apps/node_man/models.py:314 apps/node_man/serializers/ap.py:69
#: apps/node_man/serializers/host.py:44 apps/node_man/serializers/job.py:52
msgid "外网IP"
msgstr "External IP"

#: apps/node_man/models.py:315 apps/node_man/serializers/host.py:45
#: apps/node_man/serializers/job.py:53
msgid "登录IP"
msgstr "Login IP"

#: apps/node_man/models.py:316 apps/node_man/serializers/host.py:46
#: apps/node_man/serializers/job.py:54
msgid "数据IP"
msgstr "Data IP"

#: apps/node_man/models.py:325
msgid "节点来源"
msgstr "Node source"

#: apps/node_man/models.py:326
msgid "是否手动安装"
msgstr "Is manual"

#: apps/node_man/models.py:329 apps/node_man/models.py:796
#: apps/node_man/serializers/ap.py:25 apps/node_man/serializers/cloud.py:30
#: apps/node_man/serializers/host.py:49 apps/node_man/serializers/job.py:49
msgid "接入点ID"
msgstr "Access Point ID"

#: apps/node_man/models.py:330 apps/node_man/models.py:829
#: apps/node_man/serializers/install_channel.py:30
msgid "上游节点"
msgstr "Upstream node"

#: apps/node_man/models.py:335
msgid "额外数据"
msgstr "Extra data"

#: apps/node_man/models.py:361
#, python-brace-format
msgid "{bk_host_id}| 主机信息不存在"
msgstr "{bk_host_id}| host information does not exist"

#: apps/node_man/models.py:371
#, python-brace-format
msgid "{ip}|{bk_cloud_id} 主机信息不存在"
msgstr "{ip}|{bk_cloud_id} host information does not exist"

#: apps/node_man/models.py:388
msgid "主机所属云区域不存在可用Proxy"
msgstr ""
"There is no available proxy in the cloud area to which the host belongs"

#: apps/node_man/models.py:426
#, python-brace-format
msgid "安装节点主机{inner_ip}不存在，请确认是否已安装AGENT"
msgstr ""
"The installation node host {inner_ip} does not exist, please confirm whether "
"AGENT is installed"

#: apps/node_man/models.py:488 apps/node_man/models.py:489
msgid "主机（Host）"
msgstr "Host"

#: apps/node_man/models.py:500
msgid "默认"
msgstr "Default"

#: apps/node_man/models.py:507
msgid "进程名称"
msgstr "Process name"

#: apps/node_man/models.py:509
msgid "进程状态"
msgstr "Process status"

#: apps/node_man/models.py:516
msgid "是否自动启动"
msgstr "Whether to start automatically"

#: apps/node_man/models.py:518
msgid "进程版本"
msgstr "Process version"

#: apps/node_man/models.py:520
msgid "进程类型"
msgstr "Process type"

#: apps/node_man/models.py:523
msgid "配置文件"
msgstr "Config file"

#: apps/node_man/models.py:524
msgid "监听IP"
msgstr "Listening IP"

#: apps/node_man/models.py:525
msgid "监听端口"
msgstr "Listening port"

#: apps/node_man/models.py:527
msgid "二进制文件所在路径"
msgstr "Path to binary file"

#: apps/node_man/models.py:528 apps/node_man/models.py:1217
msgid "日志路径"
msgstr "Log path"

#: apps/node_man/models.py:529 apps/node_man/models.py:1218
#: apps/node_man/serializers/host.py:55 apps/node_man/serializers/job.py:65
msgid "数据文件路径"
msgstr "Data file path"

#: apps/node_man/models.py:530 apps/node_man/models.py:1219
msgid "pid文件路径"
msgstr "Pid file path"

#: apps/node_man/models.py:532
msgid "插件组ID"
msgstr "Plugin Group ID"

#: apps/node_man/models.py:534
msgid "来源类型"
msgstr "Source type"

#: apps/node_man/models.py:536
msgid "来源ID"
msgstr "Source ID"

#: apps/node_man/models.py:539 apps/node_man/models.py:1557
msgid "CMDB对象ID"
msgstr "CMDB object ID"

#: apps/node_man/models.py:540
msgid "是否是最新记录"
msgstr "Is it the latest record"

#: apps/node_man/models.py:571
msgid "获取主机失败: {}"
msgstr "Failed to get host: {}"

#: apps/node_man/models.py:588
msgid "获取插件信息失败: {}-{}-{}"
msgstr "Failed to get plugin info: {}-{}-{}"

#: apps/node_man/models.py:614 apps/node_man/models.py:615
msgid "进程状态（ProcessStatus）"
msgstr "ProcessStatus"

#: apps/node_man/models.py:619 apps/node_man/serializers/ap.py:26
msgid "接入点名称"
msgstr "Access Point Name"

#: apps/node_man/models.py:620 apps/node_man/serializers/ap.py:27
msgid "接入点类型"
msgstr "Access Point Type"

#: apps/node_man/models.py:621 apps/node_man/serializers/ap.py:28
msgid "区域id"
msgstr "Area id"

#: apps/node_man/models.py:622 apps/node_man/serializers/ap.py:29
msgid "城市id"
msgstr "City id"

#: apps/node_man/models.py:623 apps/node_man/serializers/ap.py:30
msgid "GSE BT文件服务器列表"
msgstr "GSE BT file server list"

#: apps/node_man/models.py:624 apps/node_man/serializers/ap.py:31
msgid "GSE 数据服务器列表"
msgstr "GSE Data Server List"

#: apps/node_man/models.py:625 apps/node_man/serializers/ap.py:32
msgid "GSE 任务服务器列表"
msgstr "GSE Task Server List"

#: apps/node_man/models.py:626 apps/node_man/serializers/ap.py:33
msgid "ZK服务器列表"
msgstr "ZK server list"

#: apps/node_man/models.py:627 apps/node_man/serializers/ap.py:34
#: apps/node_man/serializers/ap.py:79
msgid "ZK账号"
msgstr "ZK account"

#: apps/node_man/models.py:629 apps/node_man/serializers/ap.py:35
msgid "安装包内网地址"
msgstr "Installation package intranet address"

#: apps/node_man/models.py:630 apps/node_man/serializers/ap.py:36
msgid "安装包外网地址"
msgstr "Installation package external network address"

#: apps/node_man/models.py:632
msgid "Nginx路径"
msgstr "Nginx path"

#: apps/node_man/models.py:633 apps/node_man/serializers/ap.py:37
msgid "Agent配置信息"
msgstr "Agent configuration information"

#: apps/node_man/models.py:634 apps/node_man/serializers/ap.py:38
msgid "接入点状态"
msgstr "Access Point Status"

#: apps/node_man/models.py:635 apps/node_man/serializers/ap.py:39
#: apps/node_man/serializers/ap.py:82
msgid "接入点描述"
msgstr "Access Point Description"

#: apps/node_man/models.py:636 apps/node_man/serializers/ap.py:40
msgid "是否启用"
msgstr "Enable"

#: apps/node_man/models.py:637 apps/node_man/serializers/ap.py:41
msgid "是否默认接入点，不可删除"
msgstr "Is it the default access point, it cannot be deleted"

#: apps/node_man/models.py:638 apps/node_man/serializers/ap.py:83
msgid "接入点创建者"
msgstr "Access Point Creator"

#: apps/node_man/models.py:639
msgid "GSE端口配置"
msgstr "GSE port configuration"

#: apps/node_man/models.py:640 apps/node_man/serializers/ap.py:42
msgid "Proxy上的安装包"
msgstr "package on Proxy"

#: apps/node_man/models.py:641 apps/node_man/serializers/ap.py:87
msgid "节点管理外网回调地址"
msgstr "Node management external network callback address"

#: apps/node_man/models.py:642 apps/node_man/serializers/ap.py:88
msgid "节点管理内网回调地址"
msgstr "Node management intranet callback address"

#: apps/node_man/models.py:660
msgid "默认接入点不存在"
msgstr "Default access point does not exist"

#: apps/node_man/models.py:692
#, python-brace-format
msgid "回调地址请求失败，报错信息 -> {exc}"
msgstr "Callback address request failed, error message -> {exc}"

#: apps/node_man/models.py:694
msgid "回调地址指向站点非节点管理后台"
msgstr ""
"The callback address points to the site's non-node management background"

#: apps/node_man/models.py:719
#, python-brace-format
msgid "Ping {ip} 失败, {output}"
msgstr "Ping {ip} failed, {output}"

#: apps/node_man/models.py:721
#, python-brace-format
msgid "Ping {ip} 正常"
msgstr "Ping {ip} OK"

#: apps/node_man/models.py:732
#, python-brace-format
msgid "{download_url} 检测下载失败，目标地址没有 setup_agent.sh 文件"
msgstr ""
"{download_url} failed to detect download, the target address does not have a "
"setup_agent.sh file"

#: apps/node_man/models.py:740
#, python-brace-format
msgid "{download_url} 检测下载失败"
msgstr "{download_url} failed to detect download"

#: apps/node_man/models.py:747
#, python-brace-format
msgid "{download_url} 检测下载成功"
msgstr "{download_url} detected successful download"

#: apps/node_man/models.py:754
#, python-brace-format
msgid "回调地址 -> {url} 验证可达"
msgstr "Callback address->{url} is verified to be reachable"

#: apps/node_man/models.py:759
#, python-brace-format
msgid "回调地址 -> {url} 不正确：{err_msg}"
msgstr "Callback address->{url} is incorrect: {err_msg}"

#: apps/node_man/models.py:786 apps/node_man/models.py:787
msgid "接入点（AccessPoint）"
msgstr "AccessPoint"

#: apps/node_man/models.py:795 apps/node_man/serializers/cloud.py:29
msgid "云服务商"
msgstr "Cloud provider"

#: apps/node_man/models.py:797
msgid "云区域创建者"
msgstr "Cloud area creator"

#: apps/node_man/models.py:799
msgid "是否可见"
msgstr "Is visible"

#: apps/node_man/models.py:800 apps/utils/orm.py:70 pipeline/models.py:284
msgid "是否删除"
msgstr "Delete"

#: apps/node_man/models.py:811 apps/node_man/models.py:812
msgid "云区域（Cloud）"
msgstr "Cloud"

#: apps/node_man/models.py:826
msgid "名称"
msgstr "Name"

#: apps/node_man/models.py:828
msgid "安装通道跳板机"
msgstr "Install channel jumper"

#: apps/node_man/models.py:832 apps/node_man/models.py:833
msgid "安装通道（InstallChannel）"
msgstr "InstallChannel"

#: apps/node_man/models.py:839 apps/node_man/models.py:1034
#: apps/node_man/models.py:1803
msgid "操作人"
msgstr "Operator"

#: apps/node_man/models.py:841
msgid "作业类型"
msgstr "Job type"

#: apps/node_man/models.py:843 apps/node_man/models.py:1729
#: apps/node_man/models.py:2108 apps/node_man/models.py:2145
#: apps/node_man/models.py:2319
msgid "订阅ID"
msgstr "Subscription ID"

#: apps/node_man/models.py:844
msgid "任务ID列表"
msgstr "Task id list"

#: apps/node_man/models.py:845
msgid "创建任务时间"
msgstr "Task created time"

#: apps/node_man/models.py:846
msgid "任务结束时间"
msgstr "Task end time"

#: apps/node_man/models.py:848 apps/node_man/models.py:1436
#: apps/node_man/models.py:2157 apps/node_man/models.py:2305
msgid "任务状态"
msgstr "Task status"

#: apps/node_man/models.py:850
msgid "全局运行参数"
msgstr "Global run parameters"

#: apps/node_man/models.py:851
msgid "任务统计信息"
msgstr "Task statistics"

#: apps/node_man/models.py:852 apps/node_man/models.py:1808
msgid "业务范围"
msgstr "Business Scope"

#: apps/node_man/models.py:853
msgid "发生错误的主机"
msgstr "Bad host"

#: apps/node_man/models.py:854 apps/node_man/models.py:2114
msgid "是否为自动触发"
msgstr "Is it auto-triggered"

#: apps/node_man/models.py:857 apps/node_man/models.py:858
msgid "任务信息（Job）"
msgstr "Job Information (Job)"

#: apps/node_man/models.py:865
msgid "作业ID"
msgstr "Job ID"

#: apps/node_man/models.py:867 apps/node_man/models.py:2146
#: apps/node_man/models.py:2320 apps/node_man/serializers/iam.py:23
#: apps/node_man/serializers/plugin_v2.py:246
#: pipeline/contrib/statistics/models.py:36
#: pipeline/contrib/statistics/models.py:72 pipeline/models.py:611
msgid "实例ID"
msgstr "Instance ID"

#: apps/node_man/models.py:868
msgid "Pipeline节点ID"
msgstr "Pipeline node ID"

#: apps/node_man/models.py:870
msgid "当前步骤"
msgstr "Current step"

#: apps/node_man/models.py:876 apps/node_man/models.py:877
msgid "任务详情（JobTask）"
msgstr "JobTask"

#: apps/node_man/models.py:886 apps/node_man/models.py:1553
#: apps/node_man/models.py:1573
msgid "插件名"
msgstr "Plugin name"

#: apps/node_man/models.py:887
msgid "插件描述"
msgstr "Plugin Description"

#: apps/node_man/models.py:888
msgid "使用场景"
msgstr "Use scenario"

#: apps/node_man/models.py:889
msgid "英文插件描述"
msgstr "English plugin description"

#: apps/node_man/models.py:890
msgid "英文使用场景"
msgstr "English usage scenarios"

#: apps/node_man/models.py:891
msgid "所属范围"
msgstr "Belongs to"

#: apps/node_man/models.py:893
msgid "宿主节点类型要求"
msgstr "Host node type required"

#: apps/node_man/models.py:896
msgid "配置文件名称"
msgstr "Config file name"

#: apps/node_man/models.py:898
msgid "配置文件格式类型"
msgstr "Config file format type"

#: apps/node_man/models.py:906
msgid "是否使用数据库"
msgstr "Whether to use database"

#: apps/node_man/models.py:907
msgid "是否在成功安装agent后自动拉起"
msgstr ""
"Whether to automatically pull up the agent after successful installation of "
"the agent"

#: apps/node_man/models.py:908
msgid "是否二进制文件"
msgstr "Is it binary"

#: apps/node_man/models.py:909
msgid "是否启用插件"
msgstr "Whether plugins are enabled"

#: apps/node_man/models.py:912
msgid "部署方式"
msgstr "Deployment method"

#: apps/node_man/models.py:914
msgid "来源系统APP CODE"
msgstr "Source system APP CODE"

#: apps/node_man/models.py:916
msgid "节点管理管控插件信息"
msgstr "Node management control plugin information"

#: apps/node_man/models.py:919 apps/node_man/models.py:920
msgid "插件信息（GsePluginDesc）"
msgstr "Plugin Info (GsePluginDesc)"

#: apps/node_man/models.py:1014
msgid "压缩包名"
msgstr "Compressed package name"

#: apps/node_man/models.py:1015 apps/node_man/models.py:1574
msgid "版本号"
msgstr "Version"

#: apps/node_man/models.py:1016
msgid "所属服务"
msgstr "Owning service"

#: apps/node_man/models.py:1017 apps/node_man/models.py:1213
msgid "工程名"
msgstr "Project name"

#: apps/node_man/models.py:1018
msgid "包大小"
msgstr "Packet size"

#: apps/node_man/models.py:1019
msgid "包路径"
msgstr "Package path"

#: apps/node_man/models.py:1020
msgid "md5值"
msgstr "md5 value"

#: apps/node_man/models.py:1021
msgid "包更新时间"
msgstr "Package update time"

#: apps/node_man/models.py:1022
msgid "包创建时间"
msgstr "Package creation time"

#: apps/node_man/models.py:1023
msgid "安装包链接"
msgstr "Installation package link"

#: apps/node_man/models.py:1025 apps/node_man/models.py:1231
msgid "系统类型"
msgstr "System type"

#: apps/node_man/models.py:1032
msgid "CPU类型"
msgstr "CPU type"

#: apps/node_man/models.py:1036 apps/node_man/models.py:1582
msgid "是否已经发布版本"
msgstr "Is a version released"

#: apps/node_man/models.py:1038
msgid "插件是否可用"
msgstr "Is plugin available"

#: apps/node_man/models.py:1040 version_log/config.py:31
msgid "版本日志"
msgstr "Version log"

#: apps/node_man/models.py:1041
msgid "英文版本日志"
msgstr "English version log"

#: apps/node_man/models.py:1086
msgid "找不到可导出插件，请确认后重试"
msgstr "No exportable plugin found, please confirm and try again"

#: apps/node_man/models.py:1162
msgid "插件包不存在，请联系管理员处理"
msgstr ""
"The plugin package does not exist, please contact the administrator to deal "
"with it"

#: apps/node_man/models.py:1200 apps/node_man/models.py:1201
msgid "插件包（Packages）"
msgstr "Packages"

#: apps/node_man/models.py:1212 apps/node_man/models.py:1321
msgid "模块名"
msgstr "Module name"

#: apps/node_man/models.py:1214
msgid "记录对应的插件包ID"
msgstr "Record the corresponding plugin package ID"

#: apps/node_man/models.py:1216
msgid "安装路径"
msgstr "Install path"

#: apps/node_man/models.py:1221
msgid "启动命令"
msgstr "Start command"

#: apps/node_man/models.py:1222
msgid "停止命令"
msgstr "Stop command"

#: apps/node_man/models.py:1223
msgid "重启命令"
msgstr "Restart command"

#: apps/node_man/models.py:1224
msgid "重载命令"
msgstr "Reload command"

#: apps/node_man/models.py:1225
msgid "kill命令"
msgstr "Kill command"

#: apps/node_man/models.py:1226
msgid "进程版本查询命令"
msgstr "Process version query command"

#: apps/node_man/models.py:1227
msgid "进程健康检查命令"
msgstr "Process health check command"

#: apps/node_man/models.py:1228
msgid "调试进程命令"
msgstr "Debug process command"

#: apps/node_man/models.py:1233
msgid "实际二进制执行文件名"
msgstr "Actual binary executable filename"

#: apps/node_man/models.py:1234
msgid "插件允许使用的端口范围，格式 1,3,6-8,10-100"
msgstr "The range of ports allowed by the plugin, in the format 1,3,6-8,10-100"

#: apps/node_man/models.py:1235
msgid "是否需要托管"
msgstr "Do you need hosting"

#: apps/node_man/models.py:1254
#, python-format
msgid "无法解析的端口号：%s"
msgstr "Unresolved port number: %s"

#: apps/node_man/models.py:1259
#, python-format
msgid "不在合法范围内的端口号：%s"
msgstr "Port number not in valid range: %s"

#: apps/node_man/models.py:1290 apps/node_man/models.py:1299
#, python-format
msgid "不合法的端口范围定义格式：%s"
msgstr "Illegal port range definition format: %s"

#: apps/node_man/models.py:1303
#, python-format
msgid "端口范围字符串解析失败：%s"
msgstr "Failed to parse port range string: %s"

#: apps/node_man/models.py:1308 apps/node_man/models.py:1309
msgid "插件控制表（ProcControl）"
msgstr "Plugin Control Table (ProcControl)"

#: apps/node_man/models.py:1320
msgid "上传包文件名"
msgstr "Upload package filename"

#: apps/node_man/models.py:1323
msgid "文件上传的路径名"
msgstr "File upload pathname"

#: apps/node_man/models.py:1324
msgid "文件大小，单位Byte"
msgstr "File size, in Bytes"

#: apps/node_man/models.py:1325
msgid "文件MD5"
msgstr "File MD5"

#: apps/node_man/models.py:1327
msgid "文件上传时间"
msgstr "File upload time"

#: apps/node_man/models.py:1328
msgid "上传用户名"
msgstr "Upload Username"

#: apps/node_man/models.py:1329 apps/node_man/models.py:1442
#: apps/node_man/models.py:1586 apps/node_man/models.py:1663
msgid "来源系统app code"
msgstr "Source system app code"

#: apps/node_man/models.py:1332
msgid "文件包上传记录"
msgstr "Package upload log"

#: apps/node_man/models.py:1333
msgid "文件包上传记录表"
msgstr "Package upload record table"

#: apps/node_man/models.py:1362
#, python-brace-format
msgid "文件{file_path}不存在，请确认后重试"
msgstr "The file {file_path} does not exist, please confirm and try again"

#: apps/node_man/models.py:1395
msgid "文件迁移失败，请联系管理员协助处理"
msgstr "File migration failed, please contact administrator for assistance"

#: apps/node_man/models.py:1419
msgid "任务准备中"
msgstr "Task preparation"

#: apps/node_man/models.py:1420
msgid "任务进行中"
msgstr "Task in progress"

#: apps/node_man/models.py:1421
msgid "任务已完成"
msgstr "Task completed"

#: apps/node_man/models.py:1428
msgid "gse插件"
msgstr "Gse plugin"

#: apps/node_man/models.py:1432
msgid "下载文件类型"
msgstr "Download file type"

#: apps/node_man/models.py:1433
msgid "下载查询参数"
msgstr "Download query parameters"

#: apps/node_man/models.py:1435
msgid "打包后的文件路径"
msgstr "Packaged file path"

#: apps/node_man/models.py:1437
msgid "任务错误信息"
msgstr "Task error message"

#: apps/node_man/models.py:1439
msgid "任务创建者"
msgstr "Task creator"

#: apps/node_man/models.py:1440
msgid "下载任务创建时间"
msgstr "Download task creation time"

#: apps/node_man/models.py:1441
msgid "任务完成时间"
msgstr "Task completion time"

#: apps/node_man/models.py:1532
#, python-brace-format
msgid "任务失败: {err_msg}"
msgstr "Task failed: {err_msg}"

#: apps/node_man/models.py:1548 apps/node_man/models.py:1549
msgid "下载记录"
msgstr "Download log"

#: apps/node_man/models.py:1554 apps/node_man/serializers/plugin_v2.py:252
msgid "CPU限额"
msgstr "CPU quota"

#: apps/node_man/models.py:1555 apps/node_man/serializers/plugin_v2.py:253
msgid "内存限额"
msgstr "Memory quota"

#: apps/node_man/models.py:1558
msgid "CMDB实例ID"
msgstr "CMDB instance ID"

#: apps/node_man/models.py:1563 apps/node_man/models.py:1564
msgid "插件资源设置"
msgstr "Plugin Resource Settings"

#: apps/node_man/models.py:1575
msgid "配置模板名"
msgstr "Config template name"

#: apps/node_man/models.py:1576
msgid "配置模板版本"
msgstr "Config Template Version"

#: apps/node_man/models.py:1577
msgid "是否主配置"
msgstr "Is the main configuration"

#: apps/node_man/models.py:1579
msgid "文件格式"
msgstr "File format"

#: apps/node_man/models.py:1580
msgid "文件在该插件目录中相对路径"
msgstr "Relative path to file in this plugin directory"

#: apps/node_man/models.py:1581
msgid "配置内容"
msgstr "Config content"

#: apps/node_man/models.py:1584 apps/node_man/models.py:1661
#: pipeline/contrib/periodic_task/models.py:296 pipeline/models.py:276
#: pipeline/models.py:616
msgid "创建者"
msgstr "Created by"

#: apps/node_man/models.py:1592
msgid "CPU架构"
msgstr "CPU Architecture"

#: apps/node_man/models.py:1596
msgid "插件配置文件模板"
msgstr "Plugin configuration file template"

#: apps/node_man/models.py:1597
msgid "插件配置文件模板表"
msgstr "Plugin configuration file template table"

#: apps/node_man/models.py:1658
msgid "对应实例记录ID"
msgstr "Corresponding instance record ID"

#: apps/node_man/models.py:1659
msgid "渲染参数"
msgstr "Render parameters"

#: apps/node_man/models.py:1660
msgid "渲染参数MD5"
msgstr "Render parameter MD5"

#: apps/node_man/models.py:1722
msgid "插件配置文件实例"
msgstr "Plugin config file instance"

#: apps/node_man/models.py:1723
msgid "插件配置文件实例表"
msgstr "Plugin configuration file instance table"

#: apps/node_man/models.py:1730
msgid "顺序"
msgstr "Order"

#: apps/node_man/models.py:1731
msgid "步骤ID"
msgstr "Step ID"

#: apps/node_man/models.py:1732
msgid "步骤类型"
msgstr "Step type"

#: apps/node_man/models.py:1733
msgid "配置"
msgstr "Config"

#: apps/node_man/models.py:1734
msgid "参数"
msgstr "Parameter"

#: apps/node_man/models.py:1750 apps/node_man/models.py:1751
msgid "订阅步骤"
msgstr "Subscribe step"

#: apps/node_man/models.py:1765
msgid "服务"
msgstr "Service"

#: apps/node_man/models.py:1775
msgid "动态实例（拓扑）"
msgstr "Dynamic instance (topology)"

#: apps/node_man/models.py:1776
msgid "静态实例"
msgstr "Static instance"

#: apps/node_man/models.py:1777
msgid "服务模板"
msgstr "Service template"

#: apps/node_man/models.py:1778
msgid "集群模板"
msgstr "Cluster template"

#: apps/node_man/models.py:1789
msgid "一次性订阅"
msgstr "Once subscription"

#: apps/node_man/models.py:1794
msgid "任务名称"
msgstr "Task name"

#: apps/node_man/models.py:1796 apps/node_man/models.py:2323
msgid "对象类型"
msgstr "Object type"

#: apps/node_man/models.py:1798
msgid "节点"
msgstr "Node"

#: apps/node_man/models.py:1799
msgid "下发的目标机器"
msgstr "Destination machine issued"

#: apps/node_man/models.py:1800
msgid "所属系统"
msgstr "Owning system"

#: apps/node_man/models.py:1806 apps/node_man/serializers/policy.py:121
msgid "订阅类别"
msgstr "Subscription Category"

#: apps/node_man/models.py:1807 apps/node_man/serializers/plugin.py:211
#: apps/node_man/serializers/plugin_v2.py:175
#: apps/node_man/serializers/plugin_v2.py:200
#: apps/node_man/serializers/plugin_v2.py:251
#: apps/node_man/serializers/policy.py:37
#: apps/node_man/serializers/policy.py:127
msgid "插件名称"
msgstr "Plugin name"

#: apps/node_man/models.py:1810
msgid "父订阅ID"
msgstr "Parent subscription id"

#: apps/node_man/models.py:2101 apps/node_man/models.py:2102
msgid "订阅（Subscription）"
msgstr "Subscription"

#: apps/node_man/models.py:2109
msgid "执行范围"
msgstr "Execution scope"

#: apps/node_man/models.py:2110
msgid "不同step执行的动作名称。键值对"
msgstr "Name of the action performed by different steps. Key-value pair"

#: apps/node_man/models.py:2112
msgid "错误信息"
msgstr "Error message"

#: apps/node_man/models.py:2113
msgid "是否准备就绪"
msgstr "Ready"

#: apps/node_man/models.py:2115 apps/node_man/models.py:2149
msgid "Pipeline ID"
msgstr "Pipeline ID"

#: apps/node_man/models.py:2135 apps/node_man/models.py:2136
msgid "订阅任务"
msgstr "Subscribe tasks"

#: apps/node_man/models.py:2147 apps/node_man/models.py:2324
msgid "实例信息"
msgstr "Instance info"

#: apps/node_man/models.py:2148
msgid "步骤信息"
msgstr "Step info"

#: apps/node_man/models.py:2150
msgid "Start Pipeline ID"
msgstr "Start Pipeline ID"

#: apps/node_man/models.py:2153
msgid "是否需要清洗临时信息"
msgstr "Do you need to clean temporary information"

#: apps/node_man/models.py:2154
msgid "是否为实例最新记录"
msgstr "Is the latest record for the instance"

#: apps/node_man/models.py:2217 apps/node_man/models.py:2241
msgid "步骤ID [{}] 在该订阅配置中不存在"
msgstr "Step id [{}] does not exist in this subscription configuration"

#: apps/node_man/models.py:2269 apps/node_man/models.py:2270
msgid "订阅实例记录"
msgstr "Subscribe instance records"

#: apps/node_man/models.py:2274
msgid "作业实例ID"
msgstr "Job instance id"

#: apps/node_man/models.py:2275
msgid "订阅实例ID列表"
msgstr "Subscription instance ID list"

#: apps/node_man/models.py:2276 pipeline/contrib/statistics/models.py:21
#: pipeline/contrib/statistics/models.py:37
msgid "节点ID"
msgstr "Node ID"

#: apps/node_man/models.py:2277
msgid "作业状态"
msgstr "Job status"

#: apps/node_man/models.py:2280 apps/node_man/models.py:2281
msgid "作业平台ID映射表"
msgstr "Job platform ID mapping table"

#: apps/node_man/models.py:2285 apps/node_man/models.py:2302
msgid "订阅实例ID"
msgstr "Subscription instance ID"

#: apps/node_man/models.py:2286 apps/node_man/models.py:2307
#: pipeline/log/models.py:45
msgid "日志内容"
msgstr "Log content"

#: apps/node_man/models.py:2287 pipeline/log/models.py:44
msgid "日志等级"
msgstr "Log level"

#: apps/node_man/models.py:2291 apps/node_man/models.py:2292
msgid "订阅实例日志"
msgstr "Subscription instance log"

#: apps/node_man/models.py:2303
msgid "Pipeline原子ID"
msgstr "Pipeline Atom ID"

#: apps/node_man/models.py:2312 apps/node_man/models.py:2313
msgid "订阅实例状态表"
msgstr "Subscription instance status table"

#: apps/node_man/models.py:2321 apps/node_man/models.py:2365
msgid "事件类型"
msgstr "Event type"

#: apps/node_man/models.py:2322
msgid "动作"
msgstr "Action"

#: apps/node_man/models.py:2328 apps/node_man/models.py:2329
msgid "CMDB事件记录"
msgstr "CMDB event log"

#: apps/node_man/models.py:2337
msgid "PipelineID"
msgstr "PipelineID"

#: apps/node_man/models.py:2338
msgid "Pipeline拓扑树"
msgstr "Pipeline topology tree"

#: apps/node_man/models.py:2364
msgid "游标"
msgstr "Cursor"

#: apps/node_man/models.py:2366
msgid "资源"
msgstr "Resource"

#: apps/node_man/models.py:2367
msgid "事件详情"
msgstr "Event Details"

#: apps/node_man/models.py:2371 apps/node_man/models.py:2372
msgid "CMDB资源监听事件"
msgstr "CMDB resource listener event"

#: apps/node_man/serializers/ap.py:72
msgid "ZK IP地址"
msgstr "ZK IP address"

#: apps/node_man/serializers/ap.py:73
msgid "ZK 端口"
msgstr "ZK port"

#: apps/node_man/serializers/ap.py:80
msgid "ZK密码"
msgstr "ZK password"

#: apps/node_man/serializers/ap.py:81
msgid "Agent配置"
msgstr "Agent Configuration"

#: apps/node_man/serializers/ap.py:86
msgid "BSCP配置"
msgstr "BSCP configuration"

#: apps/node_man/serializers/base.py:34
#: apps/node_man/serializers/plugin_v2.py:208
#, python-brace-format
msgid "不存在名称为: {name} 的插件"
msgstr "No plugin with name: {name}"

#: apps/node_man/serializers/base.py:42
msgid "configs item must be contains (os, cpu_arch, version)"
msgstr "configs item must be contains (os, cpu_arch, version)"

#: apps/node_man/serializers/base.py:44
msgid "相同os & cpu_arch 组合至多选择一个包"
msgstr "Select at most one package for the same os &amp; cpu_arch combination"

#: apps/node_man/serializers/base.py:61 apps/node_man/serializers/base.py:78
msgid ""
"部署节点属性组合必须是 (bk_biz_id, bk_host_id) 或者 (bk_biz_id, bk_obj_id, "
"bk_inst_id)"
msgstr ""
"Deployment node attribute combination must be (bk_biz_id, bk_host_id) or "
"(bk_biz_id, bk_obj_id, bk_inst_id)"

#: apps/node_man/serializers/base.py:73
msgid "节点缺少 bk_biz_id 属性"
msgstr "Node is missing bk_biz_id attribute"

#: apps/node_man/serializers/base.py:83
#, python-brace-format
msgid "部署节点[{node}]不是主机实例"
msgstr "Deployment node [{node}] is not a host instance"

#: apps/node_man/serializers/base.py:87
#, python-brace-format
msgid "部署节点[{node}不是拓扑节点]"
msgstr "Deployment node [{node} is not a topology node]"

#: apps/node_man/serializers/cloud.py:20
msgid "是否返回直连区域"
msgstr "Whether to return the direct area"

#: apps/node_man/serializers/cloud.py:28
msgid "云区域名称"
msgstr "Cloud area name"

#: apps/node_man/serializers/cmdb.py:23
msgid "操作"
msgstr "Action"

#: apps/node_man/serializers/cmdb.py:48
msgid "搜索关键字"
msgstr "Search keyword"

#: apps/node_man/serializers/cmdb.py:50 apps/node_man/serializers/job.py:156
#: apps/node_man/serializers/policy.py:35
msgid "业务ID列表"
msgstr "List of business IDs"

#: apps/node_man/serializers/cmdb.py:53 apps/node_man/serializers/host.py:26
#: apps/node_man/serializers/host_v2.py:32 apps/node_man/serializers/job.py:31
#: apps/node_man/serializers/job.py:208 apps/node_man/serializers/plugin.py:201
#: apps/node_man/serializers/plugin_v2.py:178
#: apps/node_man/serializers/policy.py:91
#: apps/node_man/serializers/policy.py:115
msgid "当前页数"
msgstr "Current page number"

#: apps/node_man/serializers/cmdb.py:54 apps/node_man/serializers/host.py:27
#: apps/node_man/serializers/host_v2.py:33 apps/node_man/serializers/job.py:32
#: apps/node_man/serializers/job.py:209 apps/node_man/serializers/plugin.py:202
#: apps/node_man/serializers/plugin_v2.py:179
#: apps/node_man/serializers/policy.py:92
#: apps/node_man/serializers/policy.py:116
msgid "分页大小"
msgstr "Pagination Size"

#: apps/node_man/serializers/debug.py:16 apps/node_man/serializers/debug.py:20
msgid "订阅任务ID"
msgstr "Subscription task ID"

#: apps/node_man/serializers/host.py:23 apps/node_man/serializers/host.py:75
#: apps/node_man/serializers/host_v2.py:34 apps/node_man/serializers/job.py:150
#: apps/node_man/serializers/plugin.py:200
#: apps/node_man/serializers/plugin.py:237
#: apps/node_man/serializers/plugin_v2.py:180
msgid "跨页全选排除主机"
msgstr "Select all exclude hosts across pages"

#: apps/node_man/serializers/host.py:24 apps/node_man/serializers/host.py:74
#: apps/node_man/serializers/host_v2.py:29 apps/node_man/serializers/job.py:148
#: apps/node_man/serializers/job.py:207 apps/node_man/serializers/plugin.py:198
#: apps/node_man/serializers/plugin.py:235
#: apps/node_man/serializers/plugin_v2.py:176
#: apps/node_man/serializers/policy.py:90
#: apps/node_man/serializers/policy.py:114
msgid "搜索条件"
msgstr "Search Criteria"

#: apps/node_man/serializers/host.py:25
#: pipeline/contrib/periodic_task/models.py:299
msgid "额外信息"
msgstr "Extra info"

#: apps/node_man/serializers/host.py:28 apps/node_man/serializers/plugin.py:203
msgid "只返回IP"
msgstr "Only return IP"

#: apps/node_man/serializers/host.py:29
msgid "正在运行机器数"
msgstr "Number of running machines"

#: apps/node_man/serializers/host.py:47
msgid "账号"
msgstr "Account"

#: apps/node_man/serializers/host.py:48
msgid "端口号"
msgstr "Port"

#: apps/node_man/serializers/host.py:52
msgid "秘钥"
msgstr "Secret key"

#: apps/node_man/serializers/host.py:53 apps/node_man/serializers/job.py:63
msgid "加速设置"
msgstr "Acceleration Settings"

#: apps/node_man/serializers/host.py:54
msgid "加速"
msgstr "Speed up"

#: apps/node_man/serializers/host.py:71
msgid "是否针对代理"
msgstr "For proxy"

#: apps/node_man/serializers/host.py:80 apps/node_man/serializers/job.py:169
#: apps/node_man/serializers/plugin.py:257
msgid "跨页全选模式下不允许传bk_host_id参数."
msgstr "The bk_host_id parameter is not allowed in select-all across pages."

#: apps/node_man/serializers/host.py:83
msgid "不允许对代理执行跨页全选模式."
msgstr "Spread select all mode is not allowed for proxy."

#: apps/node_man/serializers/host.py:86 apps/node_man/serializers/job.py:171
#: apps/node_man/serializers/plugin.py:259
msgid "必须选择一种模式(【是否跨页全选】)"
msgstr "A mode must be selected ([whether to select all across pages])"

#: apps/node_man/serializers/host_v2.py:30
msgid "主机范围"
msgstr "Host range"

#: apps/node_man/serializers/host_v2.py:31
#: apps/node_man/serializers/host_v2.py:40
#: apps/node_man/serializers/plugin_v2.py:177
msgid "拓扑节点列表"
msgstr "Topology node list"

#: apps/node_man/serializers/host_v2.py:35
msgid "仅返回Agent统计状态"
msgstr "Only return Agent statistics status"

#: apps/node_man/serializers/host_v2.py:36
msgid "返回所有节点类型"
msgstr "Return all node types"

#: apps/node_man/serializers/host_v2.py:52
msgid "查询节点属性组合必须为(bk_biz_id, bk_inst_id, bk_obj_id)"
msgstr ""
"The query node attribute combination must be (bk_biz_id, bk_inst_id, "
"bk_obj_id)"

#: apps/node_man/serializers/iam.py:24 pipeline/models.py:615
msgid "实例名称"
msgstr "Instance name"

#: apps/node_man/serializers/iam.py:28
msgid "申请权限信息"
msgstr "Request permission information"

#: apps/node_man/serializers/install_channel.py:28
msgid "安装通道名称"
msgstr "Installation channel name"

#: apps/node_man/serializers/install_channel.py:29
msgid "跳板机节点"
msgstr "Springboard node"

#: apps/node_man/serializers/job.py:22 apps/node_man/serializers/policy.py:22
msgid "排序字段"
msgstr "Sort field"

#: apps/node_man/serializers/job.py:23 apps/node_man/serializers/policy.py:23
msgid "排序类型"
msgstr "Sort type"

#: apps/node_man/serializers/job.py:33 apps/node_man/serializers/policy.py:30
msgid "排序"
msgstr "Sort"

#: apps/node_man/serializers/job.py:34
msgid "隐藏自动部署任务"
msgstr "Hide automatic deployment tasks"

#: apps/node_man/serializers/job.py:36
msgid "任务类型列表"
msgstr "Task type list"

#: apps/node_man/serializers/job.py:37
msgid "操作类型列表"
msgstr "Operation type list"

#: apps/node_man/serializers/job.py:38
msgid "策略名称列表"
msgstr "List of policy names"

#: apps/node_man/serializers/job.py:41
msgid "起始时间"
msgstr "Start time"

#: apps/node_man/serializers/job.py:42
msgid "终止时间"
msgstr "Termination time"

#: apps/node_man/serializers/job.py:50
msgid "安装通道ID"
msgstr "Installation channel ID"

#: apps/node_man/serializers/job.py:57
msgid "账户"
msgstr "Account"

#: apps/node_man/serializers/job.py:61
msgid "是否手动模式"
msgstr "Whether manual mode"

#: apps/node_man/serializers/job.py:62
msgid "密码保留天数"
msgstr "Password retention days"

#: apps/node_man/serializers/job.py:64
msgid "传输限速"
msgstr "Transfer rate limit"

#: apps/node_man/serializers/job.py:81
#, python-brace-format
msgid "{op_type} 操作必须填写bk_host_id."
msgstr "{op_type} operation must fill in bk_host_id."

#: apps/node_man/serializers/job.py:87
#, python-brace-format
msgid "{op_type} 操作必须填写认证类型."
msgstr "{op_type} operation requires authentication type."

#: apps/node_man/serializers/job.py:92
msgid "密码认证方式必须填写密码"
msgstr "Password authentication method must fill in password"

#: apps/node_man/serializers/job.py:94
msgid "密钥认证方式必须上传密钥"
msgstr "Key authentication method must upload key"

#: apps/node_man/serializers/job.py:96
msgid "必须上传账号和端口"
msgstr "Must upload account and port"

#: apps/node_man/serializers/job.py:100
msgid "直连区域必须填写Ap_id."
msgstr "Ap_id must be filled in the direct connection area."

#: apps/node_man/serializers/job.py:113
msgid "主机信息"
msgstr "Host info"

#: apps/node_man/serializers/job.py:114
msgid "被替换的Proxy主机ID"
msgstr "Replaced proxy host ID"

#: apps/node_man/serializers/job.py:119
msgid "OA Ticket"
msgstr "OA Ticket"

#: apps/node_man/serializers/job.py:129
msgid "替换PROXY必须填写replace_host_id."
msgstr "replace PROXY must fill in replace_host_id."

#: apps/node_man/serializers/job.py:153 apps/node_man/serializers/plugin.py:240
msgid "操作类型,"
msgstr "Operation type,"

#: apps/node_man/serializers/job.py:155
msgid "主机ID列表"
msgstr "List of host IDs"

#: apps/node_man/serializers/job.py:164
msgid "该接口不可用于安装."
msgstr "The interface is not available for installation."

#: apps/node_man/serializers/job.py:166
msgid "该接口不可用于替换代理."
msgstr "This interface is not available for proxy replacement."

#: apps/node_man/serializers/job.py:218
msgid "任务实例ID列表"
msgstr "Task instance id list"

#: apps/node_man/serializers/job.py:222
msgid "任务实例ID"
msgstr "Task instance id"

#: apps/node_man/serializers/meta.py:22
msgid "安装P-Agent超时时间"
msgstr "Install P-Agent timeout"

#: apps/node_man/serializers/meta.py:23
msgid "安装Agent超时时间"
msgstr "Install Agent Timeout"

#: apps/node_man/serializers/meta.py:24
msgid "安装Proxy超时时间"
msgstr "Install Proxy timeout"

#: apps/node_man/serializers/meta.py:25
msgid "安装下载限速"
msgstr "Installation download speed limit"

#: apps/node_man/serializers/meta.py:26
msgid "并行安装数"
msgstr "Number of parallel installs"

#: apps/node_man/serializers/meta.py:27
msgid "节点管理日志级别"
msgstr "Node admin log level"

#: apps/node_man/serializers/plugin.py:204
msgid "仅返回概要信息(bk_host_id, bk_biz_id)"
msgstr "Only return summary information (bk_host_id, bk_biz_id)"

#: apps/node_man/serializers/plugin.py:205
msgid "是否节点详情"
msgstr "Is node details"

#: apps/node_man/serializers/plugin.py:207
msgid "是否返回Agent状态统计信息"
msgstr "Whether to return agent status statistics"

#: apps/node_man/serializers/plugin.py:212
#: pipeline/contrib/statistics/models.py:24
#: pipeline/contrib/statistics/models.py:46
msgid "插件版本"
msgstr "Plugin version"

#: apps/node_man/serializers/plugin.py:213
msgid "保留原有配置"
msgstr "Keep old configuration"

#: apps/node_man/serializers/plugin.py:214
msgid "不重启进程"
msgstr "Do not restart process"

#: apps/node_man/serializers/plugin.py:232
msgid "插件信息"
msgstr "Plugin info"

#: apps/node_man/serializers/plugin.py:234
msgid "插件信息列表"
msgstr "Plugin info list"

#: apps/node_man/serializers/plugin.py:250
msgid "插件参数 plugin_params 和 plugin_params_list 参数不能同时为空"
msgstr ""
"Plugin parameters plugin_params and plugin_params_list parameters cannot "
"both be empty"

#: apps/node_man/serializers/plugin.py:262
msgid "插件管理只允许对插件进行操作."
msgstr "Plugin management only allows operations on plugins."

#: apps/node_man/serializers/plugin.py:266
msgid "不允许选择重复的插件进行操作"
msgstr "Selecting duplicate plugins for operation is not allowed"

#: apps/node_man/serializers/plugin_v2.py:24
msgid "插件别名"
msgstr "Plugin alias"

#: apps/node_man/serializers/plugin_v2.py:165
msgid "仅支持'tgz', 'tar.gz'拓展名的文件"
msgstr "Only files with extension 'tgz', 'tar.gz' are supported"

#: apps/node_man/serializers/plugin_v2.py:184
#, python-brace-format
msgid "插件[{project}] 不存在"
msgstr "Plugin [{project}] does not exist"

#: apps/node_man/serializers/plugin_v2.py:190
msgid "配置模板ID列表"
msgstr "Config template ID list"

#: apps/node_man/serializers/plugin_v2.py:211
msgid "插件安装/更新需要传递steps"
msgstr "Plugin install/update requires passing steps"

#: apps/node_man/serializers/plugin_v2.py:217
msgid "插件名称列表"
msgstr "List of plugin names"

#: apps/node_man/serializers/plugin_v2.py:219
msgid "聚合关键字，可选：os/version/cpu_arch"
msgstr "Aggregate keyword, optional: os/version/cpu_arch"

#: apps/node_man/serializers/plugin_v2.py:233
#, python-brace-format
msgid "存在非法关键字 -> {illegal_keywords}, 可选项为 -> {optional_keys}"
msgstr ""
"Illegal keywords exist -> {illegal_keywords}, optional options are -> "
"{optional_keys}"

#: apps/node_man/serializers/plugin_v2.py:242
msgid "对象ID"
msgstr "Object ID"

#: apps/node_man/serializers/policy.py:38
msgid "模糊搜索关键字，支持策略名称、插件名称的模糊搜索"
msgstr ""
"Fuzzy search keyword, support fuzzy search of policy name and plugin name"

#: apps/node_man/serializers/policy.py:39
msgid "是否采取懒加载策略（仅返回一级节点）"
msgstr "Whether to adopt lazy loading strategy (only return first-level nodes)"

#: apps/node_man/serializers/policy.py:49
msgid "所属父策略"
msgstr "Owning parent policy"

#: apps/node_man/serializers/policy.py:61
msgid "灰度策略与父策略所部署插件不一致"
msgstr ""
"The grayscale policy is inconsistent with the plugin deployed by the parent "
"policy"

#: apps/node_man/serializers/policy.py:64
msgid "灰度策略不能作为父策略"
msgstr "Grayscale policy cannot be used as parent policy"

#: apps/node_man/serializers/policy.py:67
msgid "灰度策略仅支持静态IP目标"
msgstr "Grayscale policy only supports static IP targets"

#: apps/node_man/serializers/policy.py:73
msgid "灰度策略目标不在父策略范围内"
msgstr "Grayscale policy target is not within the scope of parent policy"

#: apps/node_man/serializers/policy.py:93
#: apps/node_man/serializers/policy.py:110
#: apps/node_man/serializers/policy.py:141
#: apps/node_man/serializers/policy.py:157
msgid "策略ID"
msgstr "Policy ID"

#: apps/node_man/serializers/policy.py:106
msgid "policy_id, scope 必须其中一个存在"
msgstr "policy_id, scope must exist"

#: apps/node_man/serializers/policy.py:143
msgid "仅停用策略"
msgstr "Deactivate policy only"

#: apps/node_man/serializers/policy.py:144
msgid "父策略ID"
msgstr "Parent Policy ID"

#: apps/node_man/serializers/policy.py:157
msgid "策略ID列表"
msgstr "List of policy IDs"

#: apps/node_man/tools/host.py:55
#, python-brace-format
msgid "密文无法解密，请检查是否按规则使用密钥加密：{err_msg}"
msgstr ""
"The ciphertext cannot be decrypted, please check whether the key is "
"encrypted according to the rules: {err_msg}"

#: apps/node_man/tools/host.py:57
msgid "密文解密失败：{err_msg"
msgstr "Ciphertext decryption failed: {err_msg"

#: apps/node_man/tools/job.py:35
msgid "节点异常"
msgstr "Node exception"

#: apps/node_man/tools/job.py:41
#, python-brace-format
msgid "正在 {node_name}"
msgstr "On {node_name}"

#: apps/node_man/tools/job.py:47
#, python-brace-format
msgid "{node_name} 失败"
msgstr "{node_name} failed"

#: apps/node_man/views/ap.py:345
#, python-brace-format
msgid "该接入点正在被云区域 {cloud_names} 使用"
msgstr "This access point is being used by cloud area {cloud_names}"

#: apps/node_man/views/meta.py:105
msgid "您没有权限修改任务配置"
msgstr "You do not have permission to modify task configuration"

#: apps/node_man/views/permission.py:50 common/api/modules/bk_node.py:19
#: config/default.py:284
msgid "节点管理"
msgstr "NodeMan"

#: apps/utils/drf.py:206
msgid "请求参数必须全部为整数"
msgstr "Request parameters must all be integers"

#: apps/utils/orm.py:71
msgid "删除时间"
msgstr "Delete time"

#: apps/utils/orm.py:72
msgid "删除者"
msgstr "Removed by"

#: common/api/base.py:185
#, python-brace-format
msgid "[{module}]API请求异常：({error_message})"
msgstr "[{module}]API request exception: ({error_message})"

#: common/api/base.py:251
msgid "返回数据格式不正确，结果格式非json."
msgstr ""
"The format of the returned data is incorrect, the format of the result is "
"not json."

#: common/api/base.py:322
#, python-brace-format
msgid "[BKAPI] {info}"
msgstr "[BKAPI] {info}"

#: common/api/models.py:34 common/api/models.py:35
msgid "【平台日志】API调用日志"
msgstr "[Platform log] API call log"

#: common/api/modules/esb.py:27
msgid "ESB"
msgstr "ESB"

#: common/api/modules/gse.py:19
msgid "管控平台"
msgstr "GSE"

#: common/api/modules/job.py:19
msgid "作业平台"
msgstr "JOB"

#: common/api/modules/sops.py:19
msgid "标准运维"
msgstr "BK-SOPS"

#: common/context_processors.py:44
msgid "节点管理 | 腾讯蓝鲸智云"
msgstr "NodeMan | Tencent Blueking"

#: pipeline/component_framework/models.py:42
#: pipeline/contrib/statistics/models.py:19
#: pipeline/contrib/statistics/models.py:35
msgid "组件编码"
msgstr "Component encoding"

#: pipeline/component_framework/models.py:43
msgid "组件版本"
msgstr "Component version"

#: pipeline/component_framework/models.py:44
msgid "组件名称"
msgstr "Component name"

#: pipeline/component_framework/models.py:45
msgid "组件是否可用"
msgstr "Is the component available"

#: pipeline/component_framework/models.py:50
#: pipeline/component_framework/models.py:51
msgid "组件 Component"
msgstr "Component Component"

#: pipeline/contrib/external_plugins/models/base.py:78
msgid "包源名"
msgstr "Package source name"

#: pipeline/contrib/external_plugins/models/base.py:79
msgid "是否是从配置文件中读取的"
msgstr "Is it read from configuration file"

#: pipeline/contrib/external_plugins/models/base.py:80
msgid "模块配置"
msgstr "Module configuration"

#: pipeline/contrib/external_plugins/models/source.py:24
msgid "文件托管仓库链接"
msgstr "File hosting repository link"

#: pipeline/contrib/external_plugins/models/source.py:25
msgid "分支名"
msgstr "Branch name"

#: pipeline/contrib/external_plugins/models/source.py:47
msgid "对象存储服务地址"
msgstr "Object storage service address"

#: pipeline/contrib/external_plugins/models/source.py:48
msgid "bucket 名"
msgstr "bucket 名"

#: pipeline/contrib/external_plugins/models/source.py:49
msgid "access key"
msgstr "access key"

#: pipeline/contrib/external_plugins/models/source.py:50
msgid "secret key"
msgstr "secret key"

#: pipeline/contrib/external_plugins/models/source.py:78
msgid "文件系统路径"
msgstr "filesystem path"

#: pipeline/contrib/periodic_task/models.py:35
msgid "Days"
msgstr "Days"

#: pipeline/contrib/periodic_task/models.py:36
msgid "Hours"
msgstr "Hours"

#: pipeline/contrib/periodic_task/models.py:37
msgid "Minutes"
msgstr "Minutes"

#: pipeline/contrib/periodic_task/models.py:38
msgid "Seconds"
msgstr "Seconds"

#: pipeline/contrib/periodic_task/models.py:39
msgid "Microseconds"
msgstr "Microseconds"

#: pipeline/contrib/periodic_task/models.py:45
msgid "every"
msgstr "every"

#: pipeline/contrib/periodic_task/models.py:46
msgid "period"
msgstr "period"

#: pipeline/contrib/periodic_task/models.py:49
#: pipeline/contrib/periodic_task/models.py:159
msgid "interval"
msgstr "interval"

#: pipeline/contrib/periodic_task/models.py:50
msgid "intervals"
msgstr "intervals"

#: pipeline/contrib/periodic_task/models.py:70
#, python-brace-format
msgid "every {0.period_singular}"
msgstr "every {0.period_singular}"

#: pipeline/contrib/periodic_task/models.py:71
#, python-brace-format
msgid "every {0.every:d} {0.period}"
msgstr "every {0.every:d} {0.period}"

#: pipeline/contrib/periodic_task/models.py:84
msgid "minute"
msgstr "minute"

#: pipeline/contrib/periodic_task/models.py:85
msgid "hour"
msgstr "hour"

#: pipeline/contrib/periodic_task/models.py:86
msgid "day of week"
msgstr "day of week"

#: pipeline/contrib/periodic_task/models.py:87
msgid "day of month"
msgstr "day of month"

#: pipeline/contrib/periodic_task/models.py:88
msgid "month of year"
msgstr "month of year"

#: pipeline/contrib/periodic_task/models.py:92
#: pipeline/contrib/periodic_task/models.py:165
msgid "crontab"
msgstr "crontab"

#: pipeline/contrib/periodic_task/models.py:93
msgid "crontabs"
msgstr "crontabs"

#: pipeline/contrib/periodic_task/models.py:156
msgid "name"
msgstr "name"

#: pipeline/contrib/periodic_task/models.py:156
msgid "Useful description"
msgstr "Useful description"

#: pipeline/contrib/periodic_task/models.py:157
msgid "task name"
msgstr "task name"

#: pipeline/contrib/periodic_task/models.py:167
msgid "Use one of interval/crontab"
msgstr "Use one of interval/crontab"

#: pipeline/contrib/periodic_task/models.py:169
msgid "Arguments"
msgstr "Arguments"

#: pipeline/contrib/periodic_task/models.py:169
msgid "JSON encoded positional arguments"
msgstr "JSON encoded positional arguments"

#: pipeline/contrib/periodic_task/models.py:171
msgid "Keyword arguments"
msgstr "Keyword arguments"

#: pipeline/contrib/periodic_task/models.py:171
msgid "JSON encoded keyword arguments"
msgstr "JSON encoded keyword arguments"

#: pipeline/contrib/periodic_task/models.py:174
msgid "queue"
msgstr "queue"

#: pipeline/contrib/periodic_task/models.py:174
msgid "Queue defined in CELERY_QUEUES"
msgstr "Queue defined in CELERY_QUEUES"

#: pipeline/contrib/periodic_task/models.py:176
msgid "exchange"
msgstr "exchange"

#: pipeline/contrib/periodic_task/models.py:177
msgid "routing key"
msgstr "routing key"

#: pipeline/contrib/periodic_task/models.py:178
msgid "expires"
msgstr "expires"

#: pipeline/contrib/periodic_task/models.py:179
msgid "enabled"
msgstr "enabled"

#: pipeline/contrib/periodic_task/models.py:183
msgid "description"
msgstr "description"

#: pipeline/contrib/periodic_task/models.py:189
msgid "djcelery periodic task"
msgstr "djcelery periodic task"

#: pipeline/contrib/periodic_task/models.py:190
msgid "djcelery periodic tasks"
msgstr "djcelery periodic tasks"

#: pipeline/contrib/periodic_task/models.py:278
msgid "周期任务名称"
msgstr "period task name"

#: pipeline/contrib/periodic_task/models.py:283
msgid "周期任务对应的模板"
msgstr "Template corresponding to periodic task"

#: pipeline/contrib/periodic_task/models.py:287
msgid "调度策略"
msgstr "Scheduling Policy"

#: pipeline/contrib/periodic_task/models.py:289
msgid "celery 周期任务实例"
msgstr "celery periodic task instance"

#: pipeline/contrib/periodic_task/models.py:292
msgid "用于创建流程实例的结构数据"
msgstr "structure data used to create process instances"

#: pipeline/contrib/periodic_task/models.py:294
msgid "执行次数"
msgstr "Executions"

#: pipeline/contrib/periodic_task/models.py:295
msgid "上次运行时间"
msgstr "Last Run"

#: pipeline/contrib/periodic_task/models.py:297
#: pipeline/contrib/periodic_task/models.py:395
#: pipeline/engine/models/core.py:594
msgid "流程优先级"
msgstr "Priority"

#: pipeline/contrib/periodic_task/models.py:298
#: pipeline/contrib/periodic_task/models.py:396
#: pipeline/engine/models/core.py:595
msgid "流程使用的队列名"
msgstr "Queue name"

#: pipeline/contrib/periodic_task/models.py:382
msgid "周期任务"
msgstr "Periodic task"

#: pipeline/contrib/periodic_task/models.py:387
msgid "Pipeline 实例"
msgstr "Pipeline instance"

#: pipeline/contrib/periodic_task/models.py:392 pipeline/log/models.py:46
msgid "异常信息"
msgstr "Exception information"

#: pipeline/contrib/periodic_task/models.py:393
#: pipeline/engine/models/core.py:883 pipeline/engine/models/core.py:990
msgid "开始时间"
msgstr "Start time"

#: pipeline/contrib/periodic_task/models.py:394
msgid "是否启动成功"
msgstr "Launch OK"

#: pipeline/contrib/statistics/models.py:20
#: pipeline/contrib/statistics/models.py:58 pipeline/models.py:273
#: pipeline/models.py:458
msgid "模板ID"
msgstr "Template ID"

#: pipeline/contrib/statistics/models.py:22
#: pipeline/contrib/statistics/models.py:38
msgid "是否子流程引用"
msgstr "sub-processes"

#: pipeline/contrib/statistics/models.py:23
#: pipeline/contrib/statistics/models.py:39
msgid "子流程堆栈"
msgstr "Stack sub-processes"

#: pipeline/contrib/statistics/models.py:23
#: pipeline/contrib/statistics/models.py:39
msgid "JSON 格式的列表"
msgstr "List JSON format"

#: pipeline/contrib/statistics/models.py:27
#: pipeline/contrib/statistics/models.py:28
msgid "Pipeline标准插件被引用数据"
msgstr "referenced data of pipeline standard plugin "

#: pipeline/contrib/statistics/models.py:40
msgid "标准插件执行开始时间"
msgstr "Standard plugin start time"

#: pipeline/contrib/statistics/models.py:41
msgid "标准插件执行结束时间"
msgstr "Standard plugin end time"

#: pipeline/contrib/statistics/models.py:42
msgid "标准插件执行耗时(s)"
msgstr "Time-consuming standard plug (s)"

#: pipeline/contrib/statistics/models.py:43
#: pipeline/core/flow/activity/service_activity.py:32
msgid "是否执行成功"
msgstr "Whether success"

#: pipeline/contrib/statistics/models.py:44 pipeline/engine/models/core.py:880
#: pipeline/engine/models/core.py:993
msgid "是否跳过"
msgstr "Whether to skip"

#: pipeline/contrib/statistics/models.py:45
msgid "是否重试记录"
msgstr "Whether to retry the record"

#: pipeline/contrib/statistics/models.py:49
#: pipeline/contrib/statistics/models.py:50
msgid "Pipeline标准插件执行数据"
msgstr "performs data of pipeline standard plugin "

#: pipeline/contrib/statistics/models.py:59
#: pipeline/contrib/statistics/models.py:73
msgid "标准插件总数"
msgstr "standard plug-ins"

#: pipeline/contrib/statistics/models.py:60
#: pipeline/contrib/statistics/models.py:74
msgid "子流程总数"
msgstr "Total number of subprocesses"

#: pipeline/contrib/statistics/models.py:61
#: pipeline/contrib/statistics/models.py:75
msgid "网关总数"
msgstr "gateway"

#: pipeline/contrib/statistics/models.py:64
#: pipeline/contrib/statistics/models.py:65
msgid "Pipeline模板引用数据"
msgstr "reference data of pipeline template"

#: pipeline/contrib/statistics/models.py:78
#: pipeline/contrib/statistics/models.py:79
msgid "Pipeline实例引用数据"
msgstr "reference data of pipeline instance"

#: pipeline/core/flow/activity/service_activity.py:32
msgid "执行结果"
msgstr "Results"

#: pipeline/core/flow/activity/service_activity.py:33
#: pipeline/engine/models/core.py:879 pipeline/engine/models/core.py:992
msgid "循环次数"
msgstr "Cycles"

#: pipeline/core/flow/activity/service_activity.py:33
msgid "循环执行次数"
msgstr "Cycles"

#: pipeline/django_signal_valve/models.py:44
msgid "信号模块名"
msgstr "Signal module name"

#: pipeline/django_signal_valve/models.py:45
msgid "信号属性名"
msgstr "Signal property name"

#: pipeline/django_signal_valve/models.py:46
msgid "信号参数"
msgstr "Signal parameters"

#: pipeline/engine/admin.py:124
msgid "打开所选的功能"
msgstr "Open the selected function"

#: pipeline/engine/admin.py:125
msgid "关闭所选的功能"
msgstr "Close selected function"

#: pipeline/engine/conf/function_switch.py:19
msgid ""
"用于冻结引擎, 冻结期间会屏蔽所有内部信号及暂停所有进程，同时拒绝所有流程控制"
"请求"
msgstr ""
"Used to freeze the engine. During freezing, all internal signals are blocked "
"and all processes are suspended.At the same time, all process control "
"requests are rejected."

#: pipeline/engine/models/core.py:53 pipeline/engine/models/core.py:616
#: pipeline/engine/models/core.py:932 pipeline/engine/models/core.py:988
#: pipeline/engine/models/core.py:1140 pipeline/engine/models/core.py:1172
#: pipeline/engine/models/core.py:1198 pipeline/engine/models/core.py:1229
#: pipeline/log/models.py:42
msgid "ID"
msgstr "ID"

#: pipeline/engine/models/core.py:54
msgid "pipeline 运行时数据"
msgstr "pipeline runtime data"

#: pipeline/engine/models/core.py:208
msgid "Process ID"
msgstr "Process ID"

#: pipeline/engine/models/core.py:209
msgid "根 pipeline 的 ID"
msgstr "Root pipeline of ID"

#: pipeline/engine/models/core.py:210
msgid "当前推进到的节点的 ID"
msgstr "To promote current node ID"

#: pipeline/engine/models/core.py:211
msgid "遇到该 ID 的节点就停止推进"
msgstr "The encounter node ID is stopped advancing"

#: pipeline/engine/models/core.py:212
msgid "父 process 的 ID"
msgstr "The parent process ID"

#: pipeline/engine/models/core.py:213
msgid "收到子节点 ACK 的数量"
msgstr "Number of sub-nodes receive an ACK"

#: pipeline/engine/models/core.py:214
msgid "需要收到的子节点 ACK 的数量"
msgstr "The number of child nodes need to receive an ACK"

#: pipeline/engine/models/core.py:215
msgid "该 process 是否还有效"
msgstr "Whether the process is also effective"

#: pipeline/engine/models/core.py:216
msgid "该 process 是否正在休眠"
msgstr "Whether the process is sleeping"

#: pipeline/engine/models/core.py:217
msgid "该 process 是否被冻结"
msgstr "Whether the process is frozen"

#: pipeline/engine/models/core.py:617
msgid "祖先 ID"
msgstr "ID ancestors"

#: pipeline/engine/models/core.py:618
msgid "后代 ID"
msgstr "ID offspring"

#: pipeline/engine/models/core.py:619
msgid "距离"
msgstr "distance"

#: pipeline/engine/models/core.py:875 pipeline/engine/models/core.py:923
#: pipeline/engine/models/core.py:1047 pipeline/engine/models/core.py:1230
#: pipeline/log/models.py:49
msgid "节点 ID"
msgstr "Node ID"

#: pipeline/engine/models/core.py:877
msgid "节点名称"
msgstr "Node Name"

#: pipeline/engine/models/core.py:878
msgid "重试次数"
msgstr "Number of retries"

#: pipeline/engine/models/core.py:881
msgid "是否出错后自动忽略"
msgstr "Whether after an error automatically ignored"

#: pipeline/engine/models/core.py:884
msgid "归档时间"
msgstr "Archive Time"

#: pipeline/engine/models/core.py:885
msgid "版本"
msgstr "version"

#: pipeline/engine/models/core.py:886
msgid "上次状态更新的时间"
msgstr "Last time status updates"

#: pipeline/engine/models/core.py:924 pipeline/engine/models/core.py:933
msgid "输入数据"
msgstr "Input data"

#: pipeline/engine/models/core.py:925 pipeline/engine/models/core.py:934
msgid "输出数据"
msgstr "Output Data"

#: pipeline/engine/models/core.py:926 pipeline/engine/models/core.py:935
msgid "异常数据"
msgstr "abnormal data"

#: pipeline/engine/models/core.py:941
msgid "自增ID"
msgstr "Increment ID"

#: pipeline/engine/models/core.py:942
msgid "回调服务ID"
msgstr "Callback Service ID"

#: pipeline/engine/models/core.py:943 pipeline/engine/models/core.py:1052
msgid "回调数据"
msgstr "Callback data"

#: pipeline/engine/models/core.py:989
msgid "节点 id"
msgstr "Node id"

#: pipeline/engine/models/core.py:991 pipeline/models.py:620
msgid "结束时间"
msgstr "End Time"

#: pipeline/engine/models/core.py:1046
msgid "ID 节点ID+version"
msgstr "ID Node ID + version"

#: pipeline/engine/models/core.py:1048
msgid "Pipeline 进程 ID"
msgstr "Pipeline Process ID"

#: pipeline/engine/models/core.py:1049
msgid "被调度次数"
msgstr "Scheduling times"

#: pipeline/engine/models/core.py:1050
msgid "是否是回调型调度"
msgstr "Whether it is scheduling a callback type"

#: pipeline/engine/models/core.py:1051
msgid "是否支持多次回调"
msgstr "Whether to support multiple callbacks"

#: pipeline/engine/models/core.py:1053
msgid "待调度服务"
msgstr "Be-scheduled services"

#: pipeline/engine/models/core.py:1054
msgid "是否已完成"
msgstr "completed or not"

#: pipeline/engine/models/core.py:1055
msgid "Activity 的版本"
msgstr "Activity version"

#: pipeline/engine/models/core.py:1056
msgid "是否正在被调度"
msgstr "Whether it is scheduled"

#: pipeline/engine/models/core.py:1141
msgid "子流程 ID"
msgstr "Sub-process ID"

#: pipeline/engine/models/core.py:1142
msgid "对应的进程 ID"
msgstr "The corresponding process ID"

#: pipeline/engine/models/core.py:1173
msgid "pipeline 进程 ID"
msgstr "pipeline process ID"

#: pipeline/engine/models/core.py:1174 pipeline/engine/models/core.py:1200
#: pipeline/engine/models/core.py:1231
msgid "celery 任务 ID"
msgstr "celery task ID"

#: pipeline/engine/models/core.py:1199
msgid "schedule ID"
msgstr "schedule ID"

#: pipeline/engine/models/data.py:45
msgid "对象唯一键"
msgstr "The only key objects"

#: pipeline/engine/models/data.py:46
msgid "对象存储字段"
msgstr "Object storage field"

#: pipeline/engine/models/function.py:54
msgid "功能名称"
msgstr "function name"

#: pipeline/engine/models/function.py:55
msgid "功能描述"
msgstr "Functional Description"

#: pipeline/engine/models/function.py:56
msgid "是否激活"
msgstr "Activate or not"

#: pipeline/log/models.py:43
msgid "logger 名称"
msgstr "logger name"

#: pipeline/log/models.py:47
msgid "输出时间"
msgstr "Output Time"

#: pipeline/log/models.py:50
msgid "节点执行历史 ID"
msgstr "Node execution history ID"

#: pipeline/models.py:77
msgid "快照字符串的md5sum"
msgstr "md5sum string of snapshots"

#: pipeline/models.py:84 pipeline/models.py:85
msgid "模板快照"
msgstr "Template Snapshot"

#: pipeline/models.py:157
#, python-format
msgid "子流程引用链中存在循环引用：%s"
msgstr "Subprocess reference exists circular reference chain: %s"

#: pipeline/models.py:274
msgid "模板名称"
msgstr "Template Name"

#: pipeline/models.py:277 pipeline/models.py:621
msgid "描述"
msgstr "description"

#: pipeline/models.py:278
msgid "修改者"
msgstr "editor"

#: pipeline/models.py:279 pipeline/models.py:505
msgid "修改时间"
msgstr "Change the time"

#: pipeline/models.py:281
msgid "模板结构数据"
msgstr "Template structure data"

#: pipeline/models.py:283
msgid "是否含有子流程"
msgstr "If it contains sub-processes"

#: pipeline/models.py:284
msgid "表示当前模板是否删除"
msgstr "Indicate whether to delete the current template"

#: pipeline/models.py:289 pipeline/models.py:290 pipeline/models.py:613
msgid "Pipeline模板"
msgstr "Pipeline Templates"

#: pipeline/models.py:432
msgid "根模板ID"
msgstr "Root template ID"

#: pipeline/models.py:433
msgid "子流程模板ID"
msgstr "Sub-process template ID"

#: pipeline/models.py:434
msgid "子流程节点 ID"
msgstr "Sub process node ID"

#: pipeline/models.py:435 pipeline/models.py:459 pipeline/models.py:489
msgid "快照字符串的md5"
msgstr "md5 snapshot string"

#: pipeline/models.py:487
msgid "模板 ID"
msgstr "Template ID"

#: pipeline/models.py:488
msgid "模板数据 ID"
msgstr "Template data ID"

#: pipeline/models.py:490
msgid "添加日期"
msgstr "Date Added"

#: pipeline/models.py:501
msgid "对应模板 ID"
msgstr "The corresponding template ID"

#: pipeline/models.py:503
msgid "方案唯一ID"
msgstr "Unique ID program"

#: pipeline/models.py:504
msgid "方案名称"
msgstr "Program Name"

#: pipeline/models.py:506
msgid "方案数据"
msgstr "Program data"

#: pipeline/models.py:619
msgid "启动时间"
msgstr "Start Time"

#: pipeline/models.py:622
msgid "是否已经启动"
msgstr "started or not"

#: pipeline/models.py:623
msgid "是否已经完成"
msgstr "completed or not"

#: pipeline/models.py:624
msgid "是否已经撤销"
msgstr "revoked or not"

#: pipeline/models.py:625
msgid "是否已经删除"
msgstr "deleted or not"

#: pipeline/models.py:625
msgid "表示当前实例是否删除"
msgstr "Indicates whether the current instance is deleted or not"

#: pipeline/models.py:631
msgid "实例结构数据，指向实例对应的模板的结构数据"
msgstr ""
"Examples of data structures, data structures point to an instance of a "
"template corresponding to"

#: pipeline/models.py:639
msgid "用于实例执行的结构数据"
msgstr "Examples of data structures for execution"

#: pipeline/models.py:647
msgid "提前计算好的一些流程结构数据"
msgstr "Some good advance computing process structure data"

#: pipeline/models.py:654 pipeline/models.py:655
msgid "Pipeline实例"
msgstr "Pipeline examples"

#: pipeline/validators/connection.py:37
#, python-format
msgid "不能连接%s类型节点\n"
msgstr "Type nodes not connected %s\n"

#: pipeline/validators/connection.py:39
#, python-format
msgid "节点的入度最大为%s，最小为%s\n"
msgstr "The maximum in degree of the node is%s, and the minimum is %s\n"

#: pipeline/validators/connection.py:41
#, python-format
msgid "节点的出度最大为%s，最小为%s\n"
msgstr "The maximum outgoing degree of a node is%s, and the minimum is %s\n"

#: pipeline/validators/gateway.py:130 pipeline/validators/gateway.py:183
msgid "并行网关中的分支网关必须将所有分支汇聚到一个汇聚网关"
msgstr ""
"Gateway Gateway parallel branches must all branches converge to a converged "
"gateway"

#: pipeline/validators/gateway.py:193
msgid "分支网关的所有分支第一个遇到的汇聚网关必须是同一个"
msgstr ""
"All branches of the first convergence gateway encountered branch gateway "
"must be the same"

#: pipeline/validators/gateway.py:203
msgid "非法网关，请检查其分支是否符合规则"
msgstr "Illegal gateway, check its compliance with the rules of the branch"

#: pipeline/validators/gateway.py:220
msgid "汇聚网关只能汇聚来自同一个并行网关的分支"
msgstr ""
"Aggregation Gateway only brought together from the same branch of a parallel "
"gateway"

#: pipeline/validators/gateway.py:225
msgid "汇聚网关没有汇聚其对应的并行网关的所有分支"
msgstr ""
"No aggregation aggregation gateway corresponding parallel gateways all "
"branches"

#: pipeline/validators/gateway.py:319
msgid "无法获取该网关距离开始节点的距离"
msgstr "Unable to get away from the gateway from the start node"

#: pipeline/variable_framework/models.py:25
msgid "变量编码"
msgstr "Variable ID"

#: pipeline/variable_framework/models.py:26
msgid "变量是否可用"
msgstr "Available or not"

#: pipeline/variable_framework/models.py:29
#: pipeline/variable_framework/models.py:30
msgid "Variable变量"
msgstr "Variable variable"

#: version_log/models.py:42
msgid "用户名"
msgstr "Username"

#: version_log/models.py:43
msgid "访问版本"
msgstr "Access Version"

#: version_log/models.py:48
msgid "版本日志访问记录"
msgstr "Version log access record"

#: version_log/templates/version_log/version_logs_page.html:27
msgid "返回"
msgstr "Return"

#: version_log/views.py:56
msgid "访问出错，请联系管理员。"
msgstr "Access error, please contact administrator."

#: version_log/views.py:57
msgid "日志列表获取成功"
msgstr "Log list fetched successfully"

#: version_log/views.py:67
msgid "日志版本文件没找到，请联系管理员"
msgstr "Log version file not found, please contact administrator"

#: version_log/views.py:69
msgid "日志详情获取成功"
msgstr "Log details fetched successfully"
